<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 4 Check data | Gene expression analysis using the DE4Rumi package</title>
  <meta name="description" content="<p>Semi-automatic analysis of RNA-seq datasets, incorporating DESeq2, PIF, RIF,
and PCIT. Designed and tested on ruminant datasets that typically include
RNA-seq data form multiple tissue regions and test between multiple pairwise
treatment comparisons.</p>" />
  <meta name="generator" content="bookdown 0.24 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 4 Check data | Gene expression analysis using the DE4Rumi package" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="<p>Semi-automatic analysis of RNA-seq datasets, incorporating DESeq2, PIF, RIF,
and PCIT. Designed and tested on ruminant datasets that typically include
RNA-seq data form multiple tissue regions and test between multiple pairwise
treatment comparisons.</p>" />
  <meta name="github-repo" content="davos-i/DE4Rumi_Instructions" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 4 Check data | Gene expression analysis using the DE4Rumi package" />
  
  <meta name="twitter:description" content="<p>Semi-automatic analysis of RNA-seq datasets, incorporating DESeq2, PIF, RIF,
and PCIT. Designed and tested on ruminant datasets that typically include
RNA-seq data form multiple tissue regions and test between multiple pairwise
treatment comparisons.</p>" />
  

<meta name="author" content="David Innes" />


<meta name="date" content="2022-01-06" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="importing-data.html"/>
<link rel="next" href="cross.html"/>
<script src="libs/header-attrs-2.11/header-attrs.js"></script>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Gene expression analysis with DE4Rumi</a></li>

<li class="divider"></li>
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Introduction</a></li>
<li class="chapter" data-level="2" data-path="project-set-up.html"><a href="project-set-up.html"><i class="fa fa-check"></i><b>2</b> Project set up</a>
<ul>
<li class="chapter" data-level="2.1" data-path="project-set-up.html"><a href="project-set-up.html#use-r-notebooks"><i class="fa fa-check"></i><b>2.1</b> Use R Notebooks</a></li>
<li class="chapter" data-level="2.2" data-path="project-set-up.html"><a href="project-set-up.html#setup-packages"><i class="fa fa-check"></i><b>2.2</b> Setup packages</a></li>
<li class="chapter" data-level="2.3" data-path="project-set-up.html"><a href="project-set-up.html#a-note-on-file-paths"><i class="fa fa-check"></i><b>2.3</b> A note on file paths</a></li>
<li class="chapter" data-level="2.4" data-path="project-set-up.html"><a href="project-set-up.html#figures-displayed-below-code-chunks"><i class="fa fa-check"></i><b>2.4</b> Figures displayed below code chunks</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="importing-data.html"><a href="importing-data.html"><i class="fa fa-check"></i><b>3</b> Importing Data</a>
<ul>
<li class="chapter" data-level="3.1" data-path="importing-data.html"><a href="importing-data.html#note-on-importing-files"><i class="fa fa-check"></i><b>3.1</b> Note on importing files</a></li>
<li class="chapter" data-level="3.2" data-path="importing-data.html"><a href="importing-data.html#import-count-data"><i class="fa fa-check"></i><b>3.2</b> Import count data</a>
<ul>
<li class="chapter" data-level="3.2.1" data-path="importing-data.html"><a href="importing-data.html#check-structure-of-data"><i class="fa fa-check"></i><b>3.2.1</b> Check structure of data</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="importing-data.html"><a href="importing-data.html#import-column-metadata"><i class="fa fa-check"></i><b>3.3</b> Import column metadata</a></li>
<li class="chapter" data-level="3.4" data-path="importing-data.html"><a href="importing-data.html#rename-columns-to-match-required-format"><i class="fa fa-check"></i><b>3.4</b> Rename columns to match required format</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="check-data.html"><a href="check-data.html"><i class="fa fa-check"></i><b>4</b> Check data</a></li>
<li class="chapter" data-level="5" data-path="cross.html"><a href="cross.html"><i class="fa fa-check"></i><b>5</b> Cross-references</a>
<ul>
<li class="chapter" data-level="5.1" data-path="cross.html"><a href="cross.html#chapters-and-sub-chapters"><i class="fa fa-check"></i><b>5.1</b> Chapters and sub-chapters</a></li>
<li class="chapter" data-level="5.2" data-path="cross.html"><a href="cross.html#captioned-figures-and-tables"><i class="fa fa-check"></i><b>5.2</b> Captioned figures and tables</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Gene expression analysis using the DE4Rumi package</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="check-data" class="section level1" number="4">
<h1><span class="header-section-number">Chapter 4</span> Check data</h1>
<p>The <strong>DE4Rumi</strong> package includes some helper functions to ensure that the column data and the counts matrix are formatted correctly.</p>
<p>In the previous step, there were some unusual looking names. This is because there is coldata information for samples that were not included in this analysis. This might be common if you have sequenced a subset of samples from a larger experiment. This is not a problem, and the following functions will handle it appropriately.</p>
<p>The first check we run is the <code>check_count_matrix()</code> function. It takes the counts data, the column data and the name of the column in the column data that contains the sample names (to match to the counts data). If there is an error, it will tell you more details in the Console of R Studio (at bottom of screen).</p>
<div class="sourceCode" id="cb9"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb9-1"><a href="check-data.html#cb9-1" aria-hidden="true" tabindex="-1"></a>cts_filtered <span class="ot">&lt;-</span> </span>
<span id="cb9-2"><a href="check-data.html#cb9-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">check_count_matrix</span>(<span class="at">count_data =</span> cts_all,</span>
<span id="cb9-3"><a href="check-data.html#cb9-3" aria-hidden="true" tabindex="-1"></a>                   <span class="at">colData =</span> coldata,</span>
<span id="cb9-4"><a href="check-data.html#cb9-4" aria-hidden="true" tabindex="-1"></a>                   <span class="at">column_with_col_names =</span> sample_names)</span></code></pre></div>
<p>In this case there was some checks that did not pass, but there was also an error which terminated the function. The critical error here was:</p>
<pre><code>## FAIL: colData has entries in column_with_col_names that do not match a column in count_data. Potentially missing data?</code></pre>
<pre><code>## Entries in colData with no matching counts data: LIV_999__NO</code></pre>
<pre><code>##  Error in check_count_matrix(count_data = cts_all, colData = coldata, column_with_col_names = sample_names) : 
##   Checks terminated. See notes in Console. Run subset_colData() and try again.</code></pre>
<p>. In the console it listed the ‘Missing samples.’ The <strong>DE4Rumi</strong> also contains the function <code>subset_colData()</code> which can automatically remove these unwanted entries in the coldata.</p>
<div class="sourceCode" id="cb13"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb13-1"><a href="check-data.html#cb13-1" aria-hidden="true" tabindex="-1"></a>coldata <span class="ot">&lt;-</span> <span class="fu">subset_colData</span>(<span class="at">count_data =</span> cts_all,</span>
<span id="cb13-2"><a href="check-data.html#cb13-2" aria-hidden="true" tabindex="-1"></a>                          <span class="at">colData =</span> coldata,</span>
<span id="cb13-3"><a href="check-data.html#cb13-3" aria-hidden="true" tabindex="-1"></a>                          <span class="at">column_with_col_names =</span> sample_names)</span></code></pre></div>
<pre><code>## Entries in colData with no matching counts data: LIV_999__NO</code></pre>
<pre><code>## Dimensions before filtering:  106   5</code></pre>
<pre><code>## Dimensions after filtering:  105   5</code></pre>
<p>The output shows that the missing samples have been removed.</p>
<div class="sourceCode" id="cb17"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb17-1"><a href="check-data.html#cb17-1" aria-hidden="true" tabindex="-1"></a>cts_filtered <span class="ot">&lt;-</span> </span>
<span id="cb17-2"><a href="check-data.html#cb17-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">check_count_matrix</span>(<span class="at">count_data =</span> cts_all,</span>
<span id="cb17-3"><a href="check-data.html#cb17-3" aria-hidden="true" tabindex="-1"></a>                   <span class="at">colData =</span> coldata,</span>
<span id="cb17-4"><a href="check-data.html#cb17-4" aria-hidden="true" tabindex="-1"></a>                   <span class="at">column_with_col_names =</span> sample_names)</span></code></pre></div>
<pre><code>## FAIL: Not all columns in count_data have matching data in colData</code></pre>
<pre><code>## Missing samples:</code></pre>
<pre><code>##  [1] &quot;LIV_30&quot; &quot;LIV_25&quot; &quot;LIV_24&quot; &quot;LIV_22&quot; &quot;LIV_20&quot; &quot;LIV_19&quot; &quot;LIV_18&quot; &quot;LIV_13&quot; &quot;LIV_11&quot; &quot;LIV_10&quot; &quot;LIV_08&quot; &quot;LIV_06&quot; &quot;LIV_05&quot; &quot;LIV_04&quot; &quot;LIV_03&quot;</code></pre>
<pre><code>## Function will now output count_data without these columns!!</code></pre>
<pre><code>## PASS: Rownames not detected</code></pre>
<pre><code>## PASS: First column name is &#39;gene_ensembl&#39;</code></pre>
<pre><code>## 
##  Dimensions of raw counts dataset BEFORE sorting:</code></pre>
<pre><code>## genes:  27607     samples:  120</code></pre>
<pre><code>## 
##  Number of samples in colData: 105</code></pre>
<pre><code>## 
##  Dimensions of raw counts dataset AFTER sorting:</code></pre>
<pre><code>## genes:  27607     samples:  105</code></pre>
<ol style="list-style-type: decimal">
<li>A table of metadata, herein referred to as colData (column data).
<ul>
<li>This typically includes treatment information for each sample, but can include any number of columns that are retained with the dataset throughout.</li>
</ul></li>
</ol>

</div>
            </section>

          </div>
        </div>
      </div>
<a href="importing-data.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="cross.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/USERNAME/REPO/edit/BRANCH/03-Import_data.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["_main.pdf", "_main.epub"],
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
