<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 5 DE Analysis | Gene expression analysis using the DE4Rumi package</title>
  <meta name="description" content="<p>Semi-automatic analysis of RNA-seq datasets, incorporating DESeq2, PIF, RIF,
and PCIT. Designed and tested on ruminant datasets that typically include
RNA-seq data form multiple tissue regions and test between multiple pairwise
treatment comparisons.</p>" />
  <meta name="generator" content="bookdown 0.24 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 5 DE Analysis | Gene expression analysis using the DE4Rumi package" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="<p>Semi-automatic analysis of RNA-seq datasets, incorporating DESeq2, PIF, RIF,
and PCIT. Designed and tested on ruminant datasets that typically include
RNA-seq data form multiple tissue regions and test between multiple pairwise
treatment comparisons.</p>" />
  <meta name="github-repo" content="davos-i/DE4Rumi_Instructions" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 5 DE Analysis | Gene expression analysis using the DE4Rumi package" />
  
  <meta name="twitter:description" content="<p>Semi-automatic analysis of RNA-seq datasets, incorporating DESeq2, PIF, RIF,
and PCIT. Designed and tested on ruminant datasets that typically include
RNA-seq data form multiple tissue regions and test between multiple pairwise
treatment comparisons.</p>" />
  

<meta name="author" content="David Innes" />


<meta name="date" content="2022-01-06" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="prepare-data.html"/>
<link rel="next" href="access-results.html"/>
<script src="libs/header-attrs-2.11/header-attrs.js"></script>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Gene expression analysis with DE4Rumi</a></li>

<li class="divider"></li>
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Introduction</a>
<ul>
<li class="chapter" data-level="1.1" data-path="index.html"><a href="index.html#installing-de4rumi-package"><i class="fa fa-check"></i><b>1.1</b> Installing DE4Rumi Package</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="project-set-up.html"><a href="project-set-up.html"><i class="fa fa-check"></i><b>2</b> Project set up</a>
<ul>
<li class="chapter" data-level="2.1" data-path="project-set-up.html"><a href="project-set-up.html#use-r-notebooks"><i class="fa fa-check"></i><b>2.1</b> Use R Notebooks</a></li>
<li class="chapter" data-level="2.2" data-path="project-set-up.html"><a href="project-set-up.html#setup-packages"><i class="fa fa-check"></i><b>2.2</b> Setup packages</a></li>
<li class="chapter" data-level="2.3" data-path="project-set-up.html"><a href="project-set-up.html#a-note-on-file-paths"><i class="fa fa-check"></i><b>2.3</b> A note on file paths</a></li>
<li class="chapter" data-level="2.4" data-path="project-set-up.html"><a href="project-set-up.html#figures-displayed-below-code-chunks"><i class="fa fa-check"></i><b>2.4</b> Figures displayed below code chunks</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="importing-data.html"><a href="importing-data.html"><i class="fa fa-check"></i><b>3</b> Importing Data</a>
<ul>
<li class="chapter" data-level="3.1" data-path="importing-data.html"><a href="importing-data.html#note-on-importing-files"><i class="fa fa-check"></i><b>3.1</b> Note on importing files</a></li>
<li class="chapter" data-level="3.2" data-path="importing-data.html"><a href="importing-data.html#import-count-data"><i class="fa fa-check"></i><b>3.2</b> Import count data</a>
<ul>
<li class="chapter" data-level="3.2.1" data-path="importing-data.html"><a href="importing-data.html#check-structure-of-data"><i class="fa fa-check"></i><b>3.2.1</b> Check structure of data</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="importing-data.html"><a href="importing-data.html#import-column-metadata"><i class="fa fa-check"></i><b>3.3</b> Import column metadata</a></li>
<li class="chapter" data-level="3.4" data-path="importing-data.html"><a href="importing-data.html#rename-columns-to-match-required-format"><i class="fa fa-check"></i><b>3.4</b> Rename columns to match required format</a></li>
<li class="chapter" data-level="3.5" data-path="importing-data.html"><a href="importing-data.html#check-data"><i class="fa fa-check"></i><b>3.5</b> Check data</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="prepare-data.html"><a href="prepare-data.html"><i class="fa fa-check"></i><b>4</b> Prepare Data</a>
<ul>
<li class="chapter" data-level="4.1" data-path="prepare-data.html"><a href="prepare-data.html#get-human-equivalant-gene-names"><i class="fa fa-check"></i><b>4.1</b> Get human equivalant gene names</a></li>
<li class="chapter" data-level="4.2" data-path="prepare-data.html"><a href="prepare-data.html#prepare-summarized-experiment-object"><i class="fa fa-check"></i><b>4.2</b> Prepare summarized experiment object</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="de-analysis.html"><a href="de-analysis.html"><i class="fa fa-check"></i><b>5</b> DE Analysis</a>
<ul>
<li class="chapter" data-level="5.1" data-path="de-analysis.html"><a href="de-analysis.html#generate-de-results"><i class="fa fa-check"></i><b>5.1</b> Generate DE results</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="access-results.html"><a href="access-results.html"><i class="fa fa-check"></i><b>6</b> 6. Access results</a>
<ul>
<li class="chapter" data-level="6.1" data-path="access-results.html"><a href="access-results.html#boxplot-of-cooks-distances"><i class="fa fa-check"></i><b>6.1</b> Boxplot of cooks distances</a></li>
<li class="chapter" data-level="6.2" data-path="access-results.html"><a href="access-results.html#pca"><i class="fa fa-check"></i><b>6.2</b> PCA</a></li>
<li class="chapter" data-level="6.3" data-path="access-results.html"><a href="access-results.html#heatmap"><i class="fa fa-check"></i><b>6.3</b> Heatmap</a></li>
<li class="chapter" data-level="6.4" data-path="access-results.html"><a href="access-results.html#sample-distances"><i class="fa fa-check"></i><b>6.4</b> Sample distances</a></li>
<li class="chapter" data-level="6.5" data-path="access-results.html"><a href="access-results.html#remove-outliers-if-needed-and-re-run-steps-de-analysis"><i class="fa fa-check"></i><b>6.5</b> Remove outliers if needed and re-run steps DE Analysis</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="functional-enrichment-analysis.html"><a href="functional-enrichment-analysis.html"><i class="fa fa-check"></i><b>7</b> Functional enrichment analysis</a></li>
<li class="chapter" data-level="8" data-path="ma-plots.html"><a href="ma-plots.html"><i class="fa fa-check"></i><b>8</b> MA Plots</a></li>
<li class="chapter" data-level="9" data-path="whole-dataset-normalisation.html"><a href="whole-dataset-normalisation.html"><i class="fa fa-check"></i><b>9</b> Whole dataset normalisation</a>
<ul>
<li class="chapter" data-level="9.1" data-path="whole-dataset-normalisation.html"><a href="whole-dataset-normalisation.html#view-cooks-distance-boxplot-for-whole-dataset"><i class="fa fa-check"></i><b>9.1</b> View Cooks distance boxplot for whole dataset</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="rif-analysis.html"><a href="rif-analysis.html"><i class="fa fa-check"></i><b>10</b> RIF Analysis</a></li>
<li class="chapter" data-level="11" data-path="pcit.html"><a href="pcit.html"><i class="fa fa-check"></i><b>11</b> PCIT</a>
<ul>
<li class="chapter" data-level="11.1" data-path="pcit.html"><a href="pcit.html#prepare-data-for-pcit-analysis"><i class="fa fa-check"></i><b>11.1</b> Prepare data for PCIT analysis</a></li>
<li class="chapter" data-level="11.2" data-path="pcit.html"><a href="pcit.html#run-pcit-analysis"><i class="fa fa-check"></i><b>11.2</b> Run PCIT Analysis</a></li>
<li class="chapter" data-level="11.3" data-path="pcit.html"><a href="pcit.html#filter-pcit-output-for-cytoscape"><i class="fa fa-check"></i><b>11.3</b> Filter PCIT output for Cytoscape</a></li>
<li class="chapter" data-level="11.4" data-path="pcit.html"><a href="pcit.html#viewing-in-cytoscape"><i class="fa fa-check"></i><b>11.4</b> Viewing in Cytoscape</a>
<ul>
<li class="chapter" data-level="11.4.1" data-path="pcit.html"><a href="pcit.html#import-pcit-output-to-cytoscape"><i class="fa fa-check"></i><b>11.4.1</b> Import PCIT output to cytoscape</a></li>
<li class="chapter" data-level="11.4.2" data-path="pcit.html"><a href="pcit.html#import-metadata-for-genes"><i class="fa fa-check"></i><b>11.4.2</b> Import metadata for genes</a></li>
<li class="chapter" data-level="11.4.3" data-path="pcit.html"><a href="pcit.html#further-details"><i class="fa fa-check"></i><b>11.4.3</b> Further Details</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="12" data-path="plot-genes-of-interest.html"><a href="plot-genes-of-interest.html"><i class="fa fa-check"></i><b>12</b> Plot genes of interest</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Gene expression analysis using the DE4Rumi package</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="de-analysis" class="section level1" number="5">
<h1><span class="header-section-number">Chapter 5</span> DE Analysis</h1>
<div id="generate-de-results" class="section level2" number="5.1">
<h2><span class="header-section-number">5.1</span> Generate DE results</h2>
<p>The <code>top_level_groups</code> argument can be filled manually e.g. c(“ARC,” “LIV”)
Otherwise, use a quick function to get all possible options and parse that:</p>
<ol style="list-style-type: decimal">
<li>Get all unique entries in Tissue_Region and assign to variable</li>
</ol>
<div class="sourceCode" id="cb38"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb38-1"><a href="de-analysis.html#cb38-1" aria-hidden="true" tabindex="-1"></a>groups_for_DE <span class="ot">&lt;-</span> seq_data<span class="sc">$</span>Tissue_Region <span class="sc">%&gt;%</span> <span class="fu">unique</span>()</span>
<span id="cb38-2"><a href="de-analysis.html#cb38-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb38-3"><a href="de-analysis.html#cb38-3" aria-hidden="true" tabindex="-1"></a><span class="co">#View output to check</span></span>
<span id="cb38-4"><a href="de-analysis.html#cb38-4" aria-hidden="true" tabindex="-1"></a><span class="fu">print</span>(groups_for_DE)</span></code></pre></div>
<pre><code>## [1] &quot;ARC&quot; &quot;LHA&quot; &quot;VMH&quot; &quot;ABO&quot; &quot;DUO&quot; &quot;RUM&quot; &quot;LIV&quot;</code></pre>
<ol start="2" style="list-style-type: decimal">
<li>Notice we use <code>top_level_groups = groups_for_DE</code>. Other arguments are also
specific to this dataset. To see details of all of these arguments type
<code>?auto_generate_DE_results</code> into the Console (at bottom of screen).</li>
</ol>
<p>This is the main step where DESeq2 is used and can take a while depending on how many samples there are.</p>
<p>This function will iterate over each level of <code>top_level_groups</code>, and will automatically detect pairwise comparisons within each <code>top_level_groups</code>. This is determined based on the column data and is specified by the names given to the funciton. See <code>?auto_generate_DE_results</code> for specific details.</p>
<p>It also is set to export a lot of output files to the directory <code>"./Outputs/"</code>. This can be turned off.</p>
<div class="sourceCode" id="cb40"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb40-1"><a href="de-analysis.html#cb40-1" aria-hidden="true" tabindex="-1"></a>DE_out <span class="ot">&lt;-</span> </span>
<span id="cb40-2"><a href="de-analysis.html#cb40-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">auto_generate_DE_results</span>(<span class="at">se_data =</span> seq_data, </span>
<span id="cb40-3"><a href="de-analysis.html#cb40-3" aria-hidden="true" tabindex="-1"></a>                           <span class="at">top_level_groups =</span> groups_for_DE,</span>
<span id="cb40-4"><a href="de-analysis.html#cb40-4" aria-hidden="true" tabindex="-1"></a>                           <span class="at">top_level_colname =</span> Tissue_Region, </span>
<span id="cb40-5"><a href="de-analysis.html#cb40-5" aria-hidden="true" tabindex="-1"></a>                           <span class="at">sample_colname =</span> sample_names, </span>
<span id="cb40-6"><a href="de-analysis.html#cb40-6" aria-hidden="true" tabindex="-1"></a>                           <span class="at">samples_to_remove =</span> <span class="cn">NA</span>,</span>
<span id="cb40-7"><a href="de-analysis.html#cb40-7" aria-hidden="true" tabindex="-1"></a>                           <span class="at">DESeq2_formula_design =</span> <span class="sc">~</span>Treatment,</span>
<span id="cb40-8"><a href="de-analysis.html#cb40-8" aria-hidden="true" tabindex="-1"></a>                           <span class="at">rowSums_filter =</span> <span class="dv">10</span>, <span class="co">#for dds filtering</span></span>
<span id="cb40-9"><a href="de-analysis.html#cb40-9" aria-hidden="true" tabindex="-1"></a>                           <span class="at">results_contrast_factor =</span> Treatment,</span>
<span id="cb40-10"><a href="de-analysis.html#cb40-10" aria-hidden="true" tabindex="-1"></a>                           <span class="at">results_combinations =</span> <span class="cn">NA</span>,</span>
<span id="cb40-11"><a href="de-analysis.html#cb40-11" aria-hidden="true" tabindex="-1"></a>                           <span class="at">use_IHW_filtering =</span> <span class="cn">TRUE</span>,</span>
<span id="cb40-12"><a href="de-analysis.html#cb40-12" aria-hidden="true" tabindex="-1"></a>                           <span class="at">alpha =</span> <span class="fl">0.05</span>,</span>
<span id="cb40-13"><a href="de-analysis.html#cb40-13" aria-hidden="true" tabindex="-1"></a>                           <span class="at">gene_annotations =</span> gene_annot,</span>
<span id="cb40-14"><a href="de-analysis.html#cb40-14" aria-hidden="true" tabindex="-1"></a>                           <span class="at">export_tables =</span> <span class="cn">TRUE</span>,</span>
<span id="cb40-15"><a href="de-analysis.html#cb40-15" aria-hidden="true" tabindex="-1"></a>                           <span class="at">export_dir =</span> <span class="st">&quot;./Outputs/&quot;</span>,</span>
<span id="cb40-16"><a href="de-analysis.html#cb40-16" aria-hidden="true" tabindex="-1"></a>                           <span class="at">whole_data_normalisation =</span> <span class="cn">FALSE</span>)</span>
<span id="cb40-17"><a href="de-analysis.html#cb40-17" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-18"><a href="de-analysis.html#cb40-18" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-19"><a href="de-analysis.html#cb40-19" aria-hidden="true" tabindex="-1"></a><span class="do">##  ******************* Start of -  ARC *******************</span></span>
<span id="cb40-20"><a href="de-analysis.html#cb40-20" aria-hidden="true" tabindex="-1"></a><span class="do">## renaming the first element in assays to &#39;counts&#39;</span></span>
<span id="cb40-21"><a href="de-analysis.html#cb40-21" aria-hidden="true" tabindex="-1"></a><span class="do">## Warning in DESeq2::DESeqDataSet(se_data0, design = DESeq2_formula_design): some</span></span>
<span id="cb40-22"><a href="de-analysis.html#cb40-22" aria-hidden="true" tabindex="-1"></a><span class="do">## variables in design formula are characters, converting to factors</span></span>
<span id="cb40-23"><a href="de-analysis.html#cb40-23" aria-hidden="true" tabindex="-1"></a><span class="do">## Beginning DESeq analysis...</span></span>
<span id="cb40-24"><a href="de-analysis.html#cb40-24" aria-hidden="true" tabindex="-1"></a><span class="do">## estimating size factors</span></span>
<span id="cb40-25"><a href="de-analysis.html#cb40-25" aria-hidden="true" tabindex="-1"></a><span class="do">## estimating dispersions</span></span>
<span id="cb40-26"><a href="de-analysis.html#cb40-26" aria-hidden="true" tabindex="-1"></a><span class="do">## gene-wise dispersion estimates</span></span>
<span id="cb40-27"><a href="de-analysis.html#cb40-27" aria-hidden="true" tabindex="-1"></a><span class="do">## mean-dispersion relationship</span></span>
<span id="cb40-28"><a href="de-analysis.html#cb40-28" aria-hidden="true" tabindex="-1"></a><span class="do">## final dispersion estimates</span></span>
<span id="cb40-29"><a href="de-analysis.html#cb40-29" aria-hidden="true" tabindex="-1"></a><span class="do">## fitting model and testing</span></span>
<span id="cb40-30"><a href="de-analysis.html#cb40-30" aria-hidden="true" tabindex="-1"></a><span class="do">## Completed DESeq analysis.</span></span>
<span id="cb40-31"><a href="de-analysis.html#cb40-31" aria-hidden="true" tabindex="-1"></a><span class="do">## Plotting cooks distance...</span></span>
<span id="cb40-32"><a href="de-analysis.html#cb40-32" aria-hidden="true" tabindex="-1"></a><span class="do">## Cooks distance plot complete.</span></span>
<span id="cb40-33"><a href="de-analysis.html#cb40-33" aria-hidden="true" tabindex="-1"></a><span class="do">## Generating pairwise DESeq2 results...</span></span>
<span id="cb40-34"><a href="de-analysis.html#cb40-34" aria-hidden="true" tabindex="-1"></a><span class="do">## [1] &quot;Contrast levels: LR, MAL, LAL&quot;</span></span>
<span id="cb40-35"><a href="de-analysis.html#cb40-35" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-36"><a href="de-analysis.html#cb40-36" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-37"><a href="de-analysis.html#cb40-37" aria-hidden="true" tabindex="-1"></a><span class="do">##  log2 fold change (MLE): Treatment LR vs MAL</span></span>
<span id="cb40-38"><a href="de-analysis.html#cb40-38" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-39"><a href="de-analysis.html#cb40-39" aria-hidden="true" tabindex="-1"></a><span class="do">## out of 19758 with nonzero total read count</span></span>
<span id="cb40-40"><a href="de-analysis.html#cb40-40" aria-hidden="true" tabindex="-1"></a><span class="do">## adjusted p-value &lt; 0.05</span></span>
<span id="cb40-41"><a href="de-analysis.html#cb40-41" aria-hidden="true" tabindex="-1"></a><span class="do">## LFC &gt; 0 (up)       : 1, 0.0051%</span></span>
<span id="cb40-42"><a href="de-analysis.html#cb40-42" aria-hidden="true" tabindex="-1"></a><span class="do">## LFC &lt; 0 (down)     : 7, 0.035%</span></span>
<span id="cb40-43"><a href="de-analysis.html#cb40-43" aria-hidden="true" tabindex="-1"></a><span class="do">## outliers [1]       : 36, 0.18%</span></span>
<span id="cb40-44"><a href="de-analysis.html#cb40-44" aria-hidden="true" tabindex="-1"></a><span class="do">## [1] see &#39;cooksCutoff&#39; argument of ?results</span></span>
<span id="cb40-45"><a href="de-analysis.html#cb40-45" aria-hidden="true" tabindex="-1"></a><span class="do">## see metadata(res)$ihwResult on hypothesis weighting</span></span>
<span id="cb40-46"><a href="de-analysis.html#cb40-46" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-47"><a href="de-analysis.html#cb40-47" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-48"><a href="de-analysis.html#cb40-48" aria-hidden="true" tabindex="-1"></a><span class="do">##  log2 fold change (MLE): Treatment LR vs LAL</span></span>
<span id="cb40-49"><a href="de-analysis.html#cb40-49" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-50"><a href="de-analysis.html#cb40-50" aria-hidden="true" tabindex="-1"></a><span class="do">## out of 19758 with nonzero total read count</span></span>
<span id="cb40-51"><a href="de-analysis.html#cb40-51" aria-hidden="true" tabindex="-1"></a><span class="do">## adjusted p-value &lt; 0.05</span></span>
<span id="cb40-52"><a href="de-analysis.html#cb40-52" aria-hidden="true" tabindex="-1"></a><span class="do">## LFC &gt; 0 (up)       : 44, 0.22%</span></span>
<span id="cb40-53"><a href="de-analysis.html#cb40-53" aria-hidden="true" tabindex="-1"></a><span class="do">## LFC &lt; 0 (down)     : 80, 0.4%</span></span>
<span id="cb40-54"><a href="de-analysis.html#cb40-54" aria-hidden="true" tabindex="-1"></a><span class="do">## outliers [1]       : 36, 0.18%</span></span>
<span id="cb40-55"><a href="de-analysis.html#cb40-55" aria-hidden="true" tabindex="-1"></a><span class="do">## [1] see &#39;cooksCutoff&#39; argument of ?results</span></span>
<span id="cb40-56"><a href="de-analysis.html#cb40-56" aria-hidden="true" tabindex="-1"></a><span class="do">## see metadata(res)$ihwResult on hypothesis weighting</span></span>
<span id="cb40-57"><a href="de-analysis.html#cb40-57" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-58"><a href="de-analysis.html#cb40-58" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-59"><a href="de-analysis.html#cb40-59" aria-hidden="true" tabindex="-1"></a><span class="do">##  log2 fold change (MLE): Treatment MAL vs LAL</span></span>
<span id="cb40-60"><a href="de-analysis.html#cb40-60" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-61"><a href="de-analysis.html#cb40-61" aria-hidden="true" tabindex="-1"></a><span class="do">## out of 19758 with nonzero total read count</span></span>
<span id="cb40-62"><a href="de-analysis.html#cb40-62" aria-hidden="true" tabindex="-1"></a><span class="do">## adjusted p-value &lt; 0.05</span></span>
<span id="cb40-63"><a href="de-analysis.html#cb40-63" aria-hidden="true" tabindex="-1"></a><span class="do">## LFC &gt; 0 (up)       : 55, 0.28%</span></span>
<span id="cb40-64"><a href="de-analysis.html#cb40-64" aria-hidden="true" tabindex="-1"></a><span class="do">## LFC &lt; 0 (down)     : 124, 0.63%</span></span>
<span id="cb40-65"><a href="de-analysis.html#cb40-65" aria-hidden="true" tabindex="-1"></a><span class="do">## outliers [1]       : 36, 0.18%</span></span>
<span id="cb40-66"><a href="de-analysis.html#cb40-66" aria-hidden="true" tabindex="-1"></a><span class="do">## [1] see &#39;cooksCutoff&#39; argument of ?results</span></span>
<span id="cb40-67"><a href="de-analysis.html#cb40-67" aria-hidden="true" tabindex="-1"></a><span class="do">## see metadata(res)$ihwResult on hypothesis weighting</span></span>
<span id="cb40-68"><a href="de-analysis.html#cb40-68" aria-hidden="true" tabindex="-1"></a><span class="do">## Results generated.</span></span>
<span id="cb40-69"><a href="de-analysis.html#cb40-69" aria-hidden="true" tabindex="-1"></a><span class="do">## Plotting MA plots (default DESeq2 style for QC)...</span></span>
<span id="cb40-70"><a href="de-analysis.html#cb40-70" aria-hidden="true" tabindex="-1"></a><span class="do">## Finished MA plots.</span></span>
<span id="cb40-71"><a href="de-analysis.html#cb40-71" aria-hidden="true" tabindex="-1"></a><span class="do">## Plotting p value histograms (for QC)...</span></span>
<span id="cb40-72"><a href="de-analysis.html#cb40-72" aria-hidden="true" tabindex="-1"></a><span class="do">## Finished plotting p value histograms.</span></span>
<span id="cb40-73"><a href="de-analysis.html#cb40-73" aria-hidden="true" tabindex="-1"></a><span class="do">## ./Outputs/ Directory exists</span></span>
<span id="cb40-74"><a href="de-analysis.html#cb40-74" aria-hidden="true" tabindex="-1"></a><span class="do">## Normalised tables exported to the sub-directory: ./Outputs/</span></span>
<span id="cb40-75"><a href="de-analysis.html#cb40-75" aria-hidden="true" tabindex="-1"></a><span class="do">## PIF calculations - Preparing normalised data...</span></span>
<span id="cb40-76"><a href="de-analysis.html#cb40-76" aria-hidden="true" tabindex="-1"></a><span class="do">## PIF calculations - Preparing normalised data...COMPLETE</span></span>
<span id="cb40-77"><a href="de-analysis.html#cb40-77" aria-hidden="true" tabindex="-1"></a><span class="do">## PIF calculations - Calculating PIF data...</span></span>
<span id="cb40-78"><a href="de-analysis.html#cb40-78" aria-hidden="true" tabindex="-1"></a><span class="do">## PIF_LRvsMAL</span></span>
<span id="cb40-79"><a href="de-analysis.html#cb40-79" aria-hidden="true" tabindex="-1"></a><span class="do">## PIF_LRvsLAL</span></span>
<span id="cb40-80"><a href="de-analysis.html#cb40-80" aria-hidden="true" tabindex="-1"></a><span class="do">## PIF_MALvsLAL</span></span>
<span id="cb40-81"><a href="de-analysis.html#cb40-81" aria-hidden="true" tabindex="-1"></a><span class="do">## PIF calculations - Calculating PIF data...COMPLETE</span></span>
<span id="cb40-82"><a href="de-analysis.html#cb40-82" aria-hidden="true" tabindex="-1"></a><span class="do">## ./Outputs/ Directory exists</span></span>
<span id="cb40-83"><a href="de-analysis.html#cb40-83" aria-hidden="true" tabindex="-1"></a><span class="do">## PIF tables exported to an .xlsx file in the sub-directory: ./Outputs/</span></span>
<span id="cb40-84"><a href="de-analysis.html#cb40-84" aria-hidden="true" tabindex="-1"></a><span class="do">## PIF calculations - </span><span class="re">END</span></span>
<span id="cb40-85"><a href="de-analysis.html#cb40-85" aria-hidden="true" tabindex="-1"></a><span class="do">## ./Outputs/ Directory exists</span></span>
<span id="cb40-86"><a href="de-analysis.html#cb40-86" aria-hidden="true" tabindex="-1"></a><span class="do">## DE tables exported to an .xlsx file in the sub-directory: ./Outputs/</span></span>
<span id="cb40-87"><a href="de-analysis.html#cb40-87" aria-hidden="true" tabindex="-1"></a><span class="do">## Preparing data for output...</span></span>
<span id="cb40-88"><a href="de-analysis.html#cb40-88" aria-hidden="true" tabindex="-1"></a><span class="do">## List output succesfully generated.</span></span>
<span id="cb40-89"><a href="de-analysis.html#cb40-89" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-90"><a href="de-analysis.html#cb40-90" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-91"><a href="de-analysis.html#cb40-91" aria-hidden="true" tabindex="-1"></a><span class="do">##  ******************* </span><span class="re">END</span><span class="do"> of -  ARC *******************</span></span>
<span id="cb40-92"><a href="de-analysis.html#cb40-92" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-93"><a href="de-analysis.html#cb40-93" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-94"><a href="de-analysis.html#cb40-94" aria-hidden="true" tabindex="-1"></a><span class="do">##  ******************* Start of -  LHA *******************</span></span>
<span id="cb40-95"><a href="de-analysis.html#cb40-95" aria-hidden="true" tabindex="-1"></a><span class="do">## renaming the first element in assays to &#39;counts&#39;</span></span>
<span id="cb40-96"><a href="de-analysis.html#cb40-96" aria-hidden="true" tabindex="-1"></a><span class="do">## Warning in DESeq2::DESeqDataSet(se_data0, design = DESeq2_formula_design): some</span></span>
<span id="cb40-97"><a href="de-analysis.html#cb40-97" aria-hidden="true" tabindex="-1"></a><span class="do">## variables in design formula are characters, converting to factors</span></span>
<span id="cb40-98"><a href="de-analysis.html#cb40-98" aria-hidden="true" tabindex="-1"></a><span class="do">## Beginning DESeq analysis...</span></span>
<span id="cb40-99"><a href="de-analysis.html#cb40-99" aria-hidden="true" tabindex="-1"></a><span class="do">## estimating size factors</span></span>
<span id="cb40-100"><a href="de-analysis.html#cb40-100" aria-hidden="true" tabindex="-1"></a><span class="do">## estimating dispersions</span></span>
<span id="cb40-101"><a href="de-analysis.html#cb40-101" aria-hidden="true" tabindex="-1"></a><span class="do">## gene-wise dispersion estimates</span></span>
<span id="cb40-102"><a href="de-analysis.html#cb40-102" aria-hidden="true" tabindex="-1"></a><span class="do">## mean-dispersion relationship</span></span>
<span id="cb40-103"><a href="de-analysis.html#cb40-103" aria-hidden="true" tabindex="-1"></a><span class="do">## final dispersion estimates</span></span>
<span id="cb40-104"><a href="de-analysis.html#cb40-104" aria-hidden="true" tabindex="-1"></a><span class="do">## fitting model and testing</span></span>
<span id="cb40-105"><a href="de-analysis.html#cb40-105" aria-hidden="true" tabindex="-1"></a><span class="do">## Completed DESeq analysis.</span></span>
<span id="cb40-106"><a href="de-analysis.html#cb40-106" aria-hidden="true" tabindex="-1"></a><span class="do">## Plotting cooks distance...</span></span>
<span id="cb40-107"><a href="de-analysis.html#cb40-107" aria-hidden="true" tabindex="-1"></a><span class="do">## Cooks distance plot complete.</span></span>
<span id="cb40-108"><a href="de-analysis.html#cb40-108" aria-hidden="true" tabindex="-1"></a><span class="do">## Generating pairwise DESeq2 results...</span></span>
<span id="cb40-109"><a href="de-analysis.html#cb40-109" aria-hidden="true" tabindex="-1"></a><span class="do">## [1] &quot;Contrast levels: LR, MAL, LAL&quot;</span></span>
<span id="cb40-110"><a href="de-analysis.html#cb40-110" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-111"><a href="de-analysis.html#cb40-111" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-112"><a href="de-analysis.html#cb40-112" aria-hidden="true" tabindex="-1"></a><span class="do">##  log2 fold change (MLE): Treatment LR vs MAL</span></span>
<span id="cb40-113"><a href="de-analysis.html#cb40-113" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-114"><a href="de-analysis.html#cb40-114" aria-hidden="true" tabindex="-1"></a><span class="do">## out of 18718 with nonzero total read count</span></span>
<span id="cb40-115"><a href="de-analysis.html#cb40-115" aria-hidden="true" tabindex="-1"></a><span class="do">## adjusted p-value &lt; 0.05</span></span>
<span id="cb40-116"><a href="de-analysis.html#cb40-116" aria-hidden="true" tabindex="-1"></a><span class="do">## LFC &gt; 0 (up)       : 2, 0.011%</span></span>
<span id="cb40-117"><a href="de-analysis.html#cb40-117" aria-hidden="true" tabindex="-1"></a><span class="do">## LFC &lt; 0 (down)     : 2, 0.011%</span></span>
<span id="cb40-118"><a href="de-analysis.html#cb40-118" aria-hidden="true" tabindex="-1"></a><span class="do">## outliers [1]       : 104, 0.56%</span></span>
<span id="cb40-119"><a href="de-analysis.html#cb40-119" aria-hidden="true" tabindex="-1"></a><span class="do">## [1] see &#39;cooksCutoff&#39; argument of ?results</span></span>
<span id="cb40-120"><a href="de-analysis.html#cb40-120" aria-hidden="true" tabindex="-1"></a><span class="do">## see metadata(res)$ihwResult on hypothesis weighting</span></span>
<span id="cb40-121"><a href="de-analysis.html#cb40-121" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-122"><a href="de-analysis.html#cb40-122" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-123"><a href="de-analysis.html#cb40-123" aria-hidden="true" tabindex="-1"></a><span class="do">##  log2 fold change (MLE): Treatment LR vs LAL</span></span>
<span id="cb40-124"><a href="de-analysis.html#cb40-124" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-125"><a href="de-analysis.html#cb40-125" aria-hidden="true" tabindex="-1"></a><span class="do">## out of 18718 with nonzero total read count</span></span>
<span id="cb40-126"><a href="de-analysis.html#cb40-126" aria-hidden="true" tabindex="-1"></a><span class="do">## adjusted p-value &lt; 0.05</span></span>
<span id="cb40-127"><a href="de-analysis.html#cb40-127" aria-hidden="true" tabindex="-1"></a><span class="do">## LFC &gt; 0 (up)       : 0, 0%</span></span>
<span id="cb40-128"><a href="de-analysis.html#cb40-128" aria-hidden="true" tabindex="-1"></a><span class="do">## LFC &lt; 0 (down)     : 0, 0%</span></span>
<span id="cb40-129"><a href="de-analysis.html#cb40-129" aria-hidden="true" tabindex="-1"></a><span class="do">## outliers [1]       : 104, 0.56%</span></span>
<span id="cb40-130"><a href="de-analysis.html#cb40-130" aria-hidden="true" tabindex="-1"></a><span class="do">## [1] see &#39;cooksCutoff&#39; argument of ?results</span></span>
<span id="cb40-131"><a href="de-analysis.html#cb40-131" aria-hidden="true" tabindex="-1"></a><span class="do">## see metadata(res)$ihwResult on hypothesis weighting</span></span>
<span id="cb40-132"><a href="de-analysis.html#cb40-132" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-133"><a href="de-analysis.html#cb40-133" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-134"><a href="de-analysis.html#cb40-134" aria-hidden="true" tabindex="-1"></a><span class="do">##  log2 fold change (MLE): Treatment MAL vs LAL</span></span>
<span id="cb40-135"><a href="de-analysis.html#cb40-135" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-136"><a href="de-analysis.html#cb40-136" aria-hidden="true" tabindex="-1"></a><span class="do">## out of 18718 with nonzero total read count</span></span>
<span id="cb40-137"><a href="de-analysis.html#cb40-137" aria-hidden="true" tabindex="-1"></a><span class="do">## adjusted p-value &lt; 0.05</span></span>
<span id="cb40-138"><a href="de-analysis.html#cb40-138" aria-hidden="true" tabindex="-1"></a><span class="do">## LFC &gt; 0 (up)       : 1, 0.0053%</span></span>
<span id="cb40-139"><a href="de-analysis.html#cb40-139" aria-hidden="true" tabindex="-1"></a><span class="do">## LFC &lt; 0 (down)     : 1, 0.0053%</span></span>
<span id="cb40-140"><a href="de-analysis.html#cb40-140" aria-hidden="true" tabindex="-1"></a><span class="do">## outliers [1]       : 104, 0.56%</span></span>
<span id="cb40-141"><a href="de-analysis.html#cb40-141" aria-hidden="true" tabindex="-1"></a><span class="do">## [1] see &#39;cooksCutoff&#39; argument of ?results</span></span>
<span id="cb40-142"><a href="de-analysis.html#cb40-142" aria-hidden="true" tabindex="-1"></a><span class="do">## see metadata(res)$ihwResult on hypothesis weighting</span></span>
<span id="cb40-143"><a href="de-analysis.html#cb40-143" aria-hidden="true" tabindex="-1"></a><span class="do">## Results generated.</span></span>
<span id="cb40-144"><a href="de-analysis.html#cb40-144" aria-hidden="true" tabindex="-1"></a><span class="do">## Plotting MA plots (default DESeq2 style for QC)...</span></span>
<span id="cb40-145"><a href="de-analysis.html#cb40-145" aria-hidden="true" tabindex="-1"></a><span class="do">## Finished MA plots.</span></span>
<span id="cb40-146"><a href="de-analysis.html#cb40-146" aria-hidden="true" tabindex="-1"></a><span class="do">## Plotting p value histograms (for QC)...</span></span>
<span id="cb40-147"><a href="de-analysis.html#cb40-147" aria-hidden="true" tabindex="-1"></a><span class="do">## Finished plotting p value histograms.</span></span>
<span id="cb40-148"><a href="de-analysis.html#cb40-148" aria-hidden="true" tabindex="-1"></a><span class="do">## ./Outputs/ Directory exists</span></span>
<span id="cb40-149"><a href="de-analysis.html#cb40-149" aria-hidden="true" tabindex="-1"></a><span class="do">## Normalised tables exported to the sub-directory: ./Outputs/</span></span>
<span id="cb40-150"><a href="de-analysis.html#cb40-150" aria-hidden="true" tabindex="-1"></a><span class="do">## PIF calculations - Preparing normalised data...</span></span>
<span id="cb40-151"><a href="de-analysis.html#cb40-151" aria-hidden="true" tabindex="-1"></a><span class="do">## PIF calculations - Preparing normalised data...COMPLETE</span></span>
<span id="cb40-152"><a href="de-analysis.html#cb40-152" aria-hidden="true" tabindex="-1"></a><span class="do">## PIF calculations - Calculating PIF data...</span></span>
<span id="cb40-153"><a href="de-analysis.html#cb40-153" aria-hidden="true" tabindex="-1"></a><span class="do">## PIF_LRvsMAL</span></span>
<span id="cb40-154"><a href="de-analysis.html#cb40-154" aria-hidden="true" tabindex="-1"></a><span class="do">## PIF_LRvsLAL</span></span>
<span id="cb40-155"><a href="de-analysis.html#cb40-155" aria-hidden="true" tabindex="-1"></a><span class="do">## PIF_MALvsLAL</span></span>
<span id="cb40-156"><a href="de-analysis.html#cb40-156" aria-hidden="true" tabindex="-1"></a><span class="do">## PIF calculations - Calculating PIF data...COMPLETE</span></span>
<span id="cb40-157"><a href="de-analysis.html#cb40-157" aria-hidden="true" tabindex="-1"></a><span class="do">## ./Outputs/ Directory exists</span></span>
<span id="cb40-158"><a href="de-analysis.html#cb40-158" aria-hidden="true" tabindex="-1"></a><span class="do">## PIF tables exported to an .xlsx file in the sub-directory: ./Outputs/</span></span>
<span id="cb40-159"><a href="de-analysis.html#cb40-159" aria-hidden="true" tabindex="-1"></a><span class="do">## PIF calculations - </span><span class="re">END</span></span>
<span id="cb40-160"><a href="de-analysis.html#cb40-160" aria-hidden="true" tabindex="-1"></a><span class="do">## ./Outputs/ Directory exists</span></span>
<span id="cb40-161"><a href="de-analysis.html#cb40-161" aria-hidden="true" tabindex="-1"></a><span class="do">## DE tables exported to an .xlsx file in the sub-directory: ./Outputs/</span></span>
<span id="cb40-162"><a href="de-analysis.html#cb40-162" aria-hidden="true" tabindex="-1"></a><span class="do">## Preparing data for output...</span></span>
<span id="cb40-163"><a href="de-analysis.html#cb40-163" aria-hidden="true" tabindex="-1"></a><span class="do">## List output succesfully generated.</span></span>
<span id="cb40-164"><a href="de-analysis.html#cb40-164" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-165"><a href="de-analysis.html#cb40-165" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-166"><a href="de-analysis.html#cb40-166" aria-hidden="true" tabindex="-1"></a><span class="do">##  ******************* </span><span class="re">END</span><span class="do"> of -  LHA *******************</span></span>
<span id="cb40-167"><a href="de-analysis.html#cb40-167" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-168"><a href="de-analysis.html#cb40-168" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-169"><a href="de-analysis.html#cb40-169" aria-hidden="true" tabindex="-1"></a><span class="do">##  ******************* Start of -  VMH *******************</span></span>
<span id="cb40-170"><a href="de-analysis.html#cb40-170" aria-hidden="true" tabindex="-1"></a><span class="do">## renaming the first element in assays to &#39;counts&#39;</span></span>
<span id="cb40-171"><a href="de-analysis.html#cb40-171" aria-hidden="true" tabindex="-1"></a><span class="do">## Warning in DESeq2::DESeqDataSet(se_data0, design = DESeq2_formula_design): some</span></span>
<span id="cb40-172"><a href="de-analysis.html#cb40-172" aria-hidden="true" tabindex="-1"></a><span class="do">## variables in design formula are characters, converting to factors</span></span>
<span id="cb40-173"><a href="de-analysis.html#cb40-173" aria-hidden="true" tabindex="-1"></a><span class="do">## Beginning DESeq analysis...</span></span>
<span id="cb40-174"><a href="de-analysis.html#cb40-174" aria-hidden="true" tabindex="-1"></a><span class="do">## estimating size factors</span></span>
<span id="cb40-175"><a href="de-analysis.html#cb40-175" aria-hidden="true" tabindex="-1"></a><span class="do">## estimating dispersions</span></span>
<span id="cb40-176"><a href="de-analysis.html#cb40-176" aria-hidden="true" tabindex="-1"></a><span class="do">## gene-wise dispersion estimates</span></span>
<span id="cb40-177"><a href="de-analysis.html#cb40-177" aria-hidden="true" tabindex="-1"></a><span class="do">## mean-dispersion relationship</span></span>
<span id="cb40-178"><a href="de-analysis.html#cb40-178" aria-hidden="true" tabindex="-1"></a><span class="do">## final dispersion estimates</span></span>
<span id="cb40-179"><a href="de-analysis.html#cb40-179" aria-hidden="true" tabindex="-1"></a><span class="do">## fitting model and testing</span></span>
<span id="cb40-180"><a href="de-analysis.html#cb40-180" aria-hidden="true" tabindex="-1"></a><span class="do">## Completed DESeq analysis.</span></span>
<span id="cb40-181"><a href="de-analysis.html#cb40-181" aria-hidden="true" tabindex="-1"></a><span class="do">## Plotting cooks distance...</span></span>
<span id="cb40-182"><a href="de-analysis.html#cb40-182" aria-hidden="true" tabindex="-1"></a><span class="do">## Cooks distance plot complete.</span></span>
<span id="cb40-183"><a href="de-analysis.html#cb40-183" aria-hidden="true" tabindex="-1"></a><span class="do">## Generating pairwise DESeq2 results...</span></span>
<span id="cb40-184"><a href="de-analysis.html#cb40-184" aria-hidden="true" tabindex="-1"></a><span class="do">## [1] &quot;Contrast levels: LR, MAL, LAL&quot;</span></span>
<span id="cb40-185"><a href="de-analysis.html#cb40-185" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-186"><a href="de-analysis.html#cb40-186" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-187"><a href="de-analysis.html#cb40-187" aria-hidden="true" tabindex="-1"></a><span class="do">##  log2 fold change (MLE): Treatment LR vs MAL</span></span>
<span id="cb40-188"><a href="de-analysis.html#cb40-188" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-189"><a href="de-analysis.html#cb40-189" aria-hidden="true" tabindex="-1"></a><span class="do">## out of 19185 with nonzero total read count</span></span>
<span id="cb40-190"><a href="de-analysis.html#cb40-190" aria-hidden="true" tabindex="-1"></a><span class="do">## adjusted p-value &lt; 0.05</span></span>
<span id="cb40-191"><a href="de-analysis.html#cb40-191" aria-hidden="true" tabindex="-1"></a><span class="do">## LFC &gt; 0 (up)       : 0, 0%</span></span>
<span id="cb40-192"><a href="de-analysis.html#cb40-192" aria-hidden="true" tabindex="-1"></a><span class="do">## LFC &lt; 0 (down)     : 1, 0.0052%</span></span>
<span id="cb40-193"><a href="de-analysis.html#cb40-193" aria-hidden="true" tabindex="-1"></a><span class="do">## outliers [1]       : 40, 0.21%</span></span>
<span id="cb40-194"><a href="de-analysis.html#cb40-194" aria-hidden="true" tabindex="-1"></a><span class="do">## [1] see &#39;cooksCutoff&#39; argument of ?results</span></span>
<span id="cb40-195"><a href="de-analysis.html#cb40-195" aria-hidden="true" tabindex="-1"></a><span class="do">## see metadata(res)$ihwResult on hypothesis weighting</span></span>
<span id="cb40-196"><a href="de-analysis.html#cb40-196" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-197"><a href="de-analysis.html#cb40-197" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-198"><a href="de-analysis.html#cb40-198" aria-hidden="true" tabindex="-1"></a><span class="do">##  log2 fold change (MLE): Treatment LR vs LAL</span></span>
<span id="cb40-199"><a href="de-analysis.html#cb40-199" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-200"><a href="de-analysis.html#cb40-200" aria-hidden="true" tabindex="-1"></a><span class="do">## out of 19185 with nonzero total read count</span></span>
<span id="cb40-201"><a href="de-analysis.html#cb40-201" aria-hidden="true" tabindex="-1"></a><span class="do">## adjusted p-value &lt; 0.05</span></span>
<span id="cb40-202"><a href="de-analysis.html#cb40-202" aria-hidden="true" tabindex="-1"></a><span class="do">## LFC &gt; 0 (up)       : 4, 0.021%</span></span>
<span id="cb40-203"><a href="de-analysis.html#cb40-203" aria-hidden="true" tabindex="-1"></a><span class="do">## LFC &lt; 0 (down)     : 7, 0.036%</span></span>
<span id="cb40-204"><a href="de-analysis.html#cb40-204" aria-hidden="true" tabindex="-1"></a><span class="do">## outliers [1]       : 40, 0.21%</span></span>
<span id="cb40-205"><a href="de-analysis.html#cb40-205" aria-hidden="true" tabindex="-1"></a><span class="do">## [1] see &#39;cooksCutoff&#39; argument of ?results</span></span>
<span id="cb40-206"><a href="de-analysis.html#cb40-206" aria-hidden="true" tabindex="-1"></a><span class="do">## see metadata(res)$ihwResult on hypothesis weighting</span></span>
<span id="cb40-207"><a href="de-analysis.html#cb40-207" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-208"><a href="de-analysis.html#cb40-208" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-209"><a href="de-analysis.html#cb40-209" aria-hidden="true" tabindex="-1"></a><span class="do">##  log2 fold change (MLE): Treatment MAL vs LAL</span></span>
<span id="cb40-210"><a href="de-analysis.html#cb40-210" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-211"><a href="de-analysis.html#cb40-211" aria-hidden="true" tabindex="-1"></a><span class="do">## out of 19185 with nonzero total read count</span></span>
<span id="cb40-212"><a href="de-analysis.html#cb40-212" aria-hidden="true" tabindex="-1"></a><span class="do">## adjusted p-value &lt; 0.05</span></span>
<span id="cb40-213"><a href="de-analysis.html#cb40-213" aria-hidden="true" tabindex="-1"></a><span class="do">## LFC &gt; 0 (up)       : 1, 0.0052%</span></span>
<span id="cb40-214"><a href="de-analysis.html#cb40-214" aria-hidden="true" tabindex="-1"></a><span class="do">## LFC &lt; 0 (down)     : 4, 0.021%</span></span>
<span id="cb40-215"><a href="de-analysis.html#cb40-215" aria-hidden="true" tabindex="-1"></a><span class="do">## outliers [1]       : 40, 0.21%</span></span>
<span id="cb40-216"><a href="de-analysis.html#cb40-216" aria-hidden="true" tabindex="-1"></a><span class="do">## [1] see &#39;cooksCutoff&#39; argument of ?results</span></span>
<span id="cb40-217"><a href="de-analysis.html#cb40-217" aria-hidden="true" tabindex="-1"></a><span class="do">## see metadata(res)$ihwResult on hypothesis weighting</span></span>
<span id="cb40-218"><a href="de-analysis.html#cb40-218" aria-hidden="true" tabindex="-1"></a><span class="do">## Results generated.</span></span>
<span id="cb40-219"><a href="de-analysis.html#cb40-219" aria-hidden="true" tabindex="-1"></a><span class="do">## Plotting MA plots (default DESeq2 style for QC)...</span></span>
<span id="cb40-220"><a href="de-analysis.html#cb40-220" aria-hidden="true" tabindex="-1"></a><span class="do">## Finished MA plots.</span></span>
<span id="cb40-221"><a href="de-analysis.html#cb40-221" aria-hidden="true" tabindex="-1"></a><span class="do">## Plotting p value histograms (for QC)...</span></span>
<span id="cb40-222"><a href="de-analysis.html#cb40-222" aria-hidden="true" tabindex="-1"></a><span class="do">## Finished plotting p value histograms.</span></span>
<span id="cb40-223"><a href="de-analysis.html#cb40-223" aria-hidden="true" tabindex="-1"></a><span class="do">## ./Outputs/ Directory exists</span></span>
<span id="cb40-224"><a href="de-analysis.html#cb40-224" aria-hidden="true" tabindex="-1"></a><span class="do">## Normalised tables exported to the sub-directory: ./Outputs/</span></span>
<span id="cb40-225"><a href="de-analysis.html#cb40-225" aria-hidden="true" tabindex="-1"></a><span class="do">## PIF calculations - Preparing normalised data...</span></span>
<span id="cb40-226"><a href="de-analysis.html#cb40-226" aria-hidden="true" tabindex="-1"></a><span class="do">## PIF calculations - Preparing normalised data...COMPLETE</span></span>
<span id="cb40-227"><a href="de-analysis.html#cb40-227" aria-hidden="true" tabindex="-1"></a><span class="do">## PIF calculations - Calculating PIF data...</span></span>
<span id="cb40-228"><a href="de-analysis.html#cb40-228" aria-hidden="true" tabindex="-1"></a><span class="do">## PIF_LRvsMAL</span></span>
<span id="cb40-229"><a href="de-analysis.html#cb40-229" aria-hidden="true" tabindex="-1"></a><span class="do">## PIF_LRvsLAL</span></span>
<span id="cb40-230"><a href="de-analysis.html#cb40-230" aria-hidden="true" tabindex="-1"></a><span class="do">## PIF_MALvsLAL</span></span>
<span id="cb40-231"><a href="de-analysis.html#cb40-231" aria-hidden="true" tabindex="-1"></a><span class="do">## PIF calculations - Calculating PIF data...COMPLETE</span></span>
<span id="cb40-232"><a href="de-analysis.html#cb40-232" aria-hidden="true" tabindex="-1"></a><span class="do">## ./Outputs/ Directory exists</span></span>
<span id="cb40-233"><a href="de-analysis.html#cb40-233" aria-hidden="true" tabindex="-1"></a><span class="do">## PIF tables exported to an .xlsx file in the sub-directory: ./Outputs/</span></span>
<span id="cb40-234"><a href="de-analysis.html#cb40-234" aria-hidden="true" tabindex="-1"></a><span class="do">## PIF calculations - </span><span class="re">END</span></span>
<span id="cb40-235"><a href="de-analysis.html#cb40-235" aria-hidden="true" tabindex="-1"></a><span class="do">## ./Outputs/ Directory exists</span></span>
<span id="cb40-236"><a href="de-analysis.html#cb40-236" aria-hidden="true" tabindex="-1"></a><span class="do">## DE tables exported to an .xlsx file in the sub-directory: ./Outputs/</span></span>
<span id="cb40-237"><a href="de-analysis.html#cb40-237" aria-hidden="true" tabindex="-1"></a><span class="do">## Preparing data for output...</span></span>
<span id="cb40-238"><a href="de-analysis.html#cb40-238" aria-hidden="true" tabindex="-1"></a><span class="do">## List output succesfully generated.</span></span>
<span id="cb40-239"><a href="de-analysis.html#cb40-239" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-240"><a href="de-analysis.html#cb40-240" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-241"><a href="de-analysis.html#cb40-241" aria-hidden="true" tabindex="-1"></a><span class="do">##  ******************* </span><span class="re">END</span><span class="do"> of -  VMH *******************</span></span>
<span id="cb40-242"><a href="de-analysis.html#cb40-242" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-243"><a href="de-analysis.html#cb40-243" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-244"><a href="de-analysis.html#cb40-244" aria-hidden="true" tabindex="-1"></a><span class="do">##  ******************* Start of -  ABO *******************</span></span>
<span id="cb40-245"><a href="de-analysis.html#cb40-245" aria-hidden="true" tabindex="-1"></a><span class="do">## renaming the first element in assays to &#39;counts&#39;</span></span>
<span id="cb40-246"><a href="de-analysis.html#cb40-246" aria-hidden="true" tabindex="-1"></a><span class="do">## Warning in DESeq2::DESeqDataSet(se_data0, design = DESeq2_formula_design): some</span></span>
<span id="cb40-247"><a href="de-analysis.html#cb40-247" aria-hidden="true" tabindex="-1"></a><span class="do">## variables in design formula are characters, converting to factors</span></span>
<span id="cb40-248"><a href="de-analysis.html#cb40-248" aria-hidden="true" tabindex="-1"></a><span class="do">## Beginning DESeq analysis...</span></span>
<span id="cb40-249"><a href="de-analysis.html#cb40-249" aria-hidden="true" tabindex="-1"></a><span class="do">## estimating size factors</span></span>
<span id="cb40-250"><a href="de-analysis.html#cb40-250" aria-hidden="true" tabindex="-1"></a><span class="do">## estimating dispersions</span></span>
<span id="cb40-251"><a href="de-analysis.html#cb40-251" aria-hidden="true" tabindex="-1"></a><span class="do">## gene-wise dispersion estimates</span></span>
<span id="cb40-252"><a href="de-analysis.html#cb40-252" aria-hidden="true" tabindex="-1"></a><span class="do">## mean-dispersion relationship</span></span>
<span id="cb40-253"><a href="de-analysis.html#cb40-253" aria-hidden="true" tabindex="-1"></a><span class="do">## final dispersion estimates</span></span>
<span id="cb40-254"><a href="de-analysis.html#cb40-254" aria-hidden="true" tabindex="-1"></a><span class="do">## fitting model and testing</span></span>
<span id="cb40-255"><a href="de-analysis.html#cb40-255" aria-hidden="true" tabindex="-1"></a><span class="do">## Completed DESeq analysis.</span></span>
<span id="cb40-256"><a href="de-analysis.html#cb40-256" aria-hidden="true" tabindex="-1"></a><span class="do">## Plotting cooks distance...</span></span>
<span id="cb40-257"><a href="de-analysis.html#cb40-257" aria-hidden="true" tabindex="-1"></a><span class="do">## Cooks distance plot complete.</span></span>
<span id="cb40-258"><a href="de-analysis.html#cb40-258" aria-hidden="true" tabindex="-1"></a><span class="do">## Generating pairwise DESeq2 results...</span></span>
<span id="cb40-259"><a href="de-analysis.html#cb40-259" aria-hidden="true" tabindex="-1"></a><span class="do">## [1] &quot;Contrast levels: LR, MAL, LAL&quot;</span></span>
<span id="cb40-260"><a href="de-analysis.html#cb40-260" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-261"><a href="de-analysis.html#cb40-261" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-262"><a href="de-analysis.html#cb40-262" aria-hidden="true" tabindex="-1"></a><span class="do">##  log2 fold change (MLE): Treatment LR vs MAL</span></span>
<span id="cb40-263"><a href="de-analysis.html#cb40-263" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-264"><a href="de-analysis.html#cb40-264" aria-hidden="true" tabindex="-1"></a><span class="do">## out of 18134 with nonzero total read count</span></span>
<span id="cb40-265"><a href="de-analysis.html#cb40-265" aria-hidden="true" tabindex="-1"></a><span class="do">## adjusted p-value &lt; 0.05</span></span>
<span id="cb40-266"><a href="de-analysis.html#cb40-266" aria-hidden="true" tabindex="-1"></a><span class="do">## LFC &gt; 0 (up)       : 2, 0.011%</span></span>
<span id="cb40-267"><a href="de-analysis.html#cb40-267" aria-hidden="true" tabindex="-1"></a><span class="do">## LFC &lt; 0 (down)     : 0, 0%</span></span>
<span id="cb40-268"><a href="de-analysis.html#cb40-268" aria-hidden="true" tabindex="-1"></a><span class="do">## outliers [1]       : 230, 1.3%</span></span>
<span id="cb40-269"><a href="de-analysis.html#cb40-269" aria-hidden="true" tabindex="-1"></a><span class="do">## [1] see &#39;cooksCutoff&#39; argument of ?results</span></span>
<span id="cb40-270"><a href="de-analysis.html#cb40-270" aria-hidden="true" tabindex="-1"></a><span class="do">## see metadata(res)$ihwResult on hypothesis weighting</span></span>
<span id="cb40-271"><a href="de-analysis.html#cb40-271" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-272"><a href="de-analysis.html#cb40-272" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-273"><a href="de-analysis.html#cb40-273" aria-hidden="true" tabindex="-1"></a><span class="do">##  log2 fold change (MLE): Treatment LR vs LAL</span></span>
<span id="cb40-274"><a href="de-analysis.html#cb40-274" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-275"><a href="de-analysis.html#cb40-275" aria-hidden="true" tabindex="-1"></a><span class="do">## out of 18134 with nonzero total read count</span></span>
<span id="cb40-276"><a href="de-analysis.html#cb40-276" aria-hidden="true" tabindex="-1"></a><span class="do">## adjusted p-value &lt; 0.05</span></span>
<span id="cb40-277"><a href="de-analysis.html#cb40-277" aria-hidden="true" tabindex="-1"></a><span class="do">## LFC &gt; 0 (up)       : 9, 0.05%</span></span>
<span id="cb40-278"><a href="de-analysis.html#cb40-278" aria-hidden="true" tabindex="-1"></a><span class="do">## LFC &lt; 0 (down)     : 10, 0.055%</span></span>
<span id="cb40-279"><a href="de-analysis.html#cb40-279" aria-hidden="true" tabindex="-1"></a><span class="do">## outliers [1]       : 230, 1.3%</span></span>
<span id="cb40-280"><a href="de-analysis.html#cb40-280" aria-hidden="true" tabindex="-1"></a><span class="do">## [1] see &#39;cooksCutoff&#39; argument of ?results</span></span>
<span id="cb40-281"><a href="de-analysis.html#cb40-281" aria-hidden="true" tabindex="-1"></a><span class="do">## see metadata(res)$ihwResult on hypothesis weighting</span></span>
<span id="cb40-282"><a href="de-analysis.html#cb40-282" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-283"><a href="de-analysis.html#cb40-283" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-284"><a href="de-analysis.html#cb40-284" aria-hidden="true" tabindex="-1"></a><span class="do">##  log2 fold change (MLE): Treatment MAL vs LAL</span></span>
<span id="cb40-285"><a href="de-analysis.html#cb40-285" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-286"><a href="de-analysis.html#cb40-286" aria-hidden="true" tabindex="-1"></a><span class="do">## out of 18134 with nonzero total read count</span></span>
<span id="cb40-287"><a href="de-analysis.html#cb40-287" aria-hidden="true" tabindex="-1"></a><span class="do">## adjusted p-value &lt; 0.05</span></span>
<span id="cb40-288"><a href="de-analysis.html#cb40-288" aria-hidden="true" tabindex="-1"></a><span class="do">## LFC &gt; 0 (up)       : 13, 0.072%</span></span>
<span id="cb40-289"><a href="de-analysis.html#cb40-289" aria-hidden="true" tabindex="-1"></a><span class="do">## LFC &lt; 0 (down)     : 8, 0.044%</span></span>
<span id="cb40-290"><a href="de-analysis.html#cb40-290" aria-hidden="true" tabindex="-1"></a><span class="do">## outliers [1]       : 230, 1.3%</span></span>
<span id="cb40-291"><a href="de-analysis.html#cb40-291" aria-hidden="true" tabindex="-1"></a><span class="do">## [1] see &#39;cooksCutoff&#39; argument of ?results</span></span>
<span id="cb40-292"><a href="de-analysis.html#cb40-292" aria-hidden="true" tabindex="-1"></a><span class="do">## see metadata(res)$ihwResult on hypothesis weighting</span></span>
<span id="cb40-293"><a href="de-analysis.html#cb40-293" aria-hidden="true" tabindex="-1"></a><span class="do">## Results generated.</span></span>
<span id="cb40-294"><a href="de-analysis.html#cb40-294" aria-hidden="true" tabindex="-1"></a><span class="do">## Plotting MA plots (default DESeq2 style for QC)...</span></span>
<span id="cb40-295"><a href="de-analysis.html#cb40-295" aria-hidden="true" tabindex="-1"></a><span class="do">## Finished MA plots.</span></span>
<span id="cb40-296"><a href="de-analysis.html#cb40-296" aria-hidden="true" tabindex="-1"></a><span class="do">## Plotting p value histograms (for QC)...</span></span>
<span id="cb40-297"><a href="de-analysis.html#cb40-297" aria-hidden="true" tabindex="-1"></a><span class="do">## Finished plotting p value histograms.</span></span>
<span id="cb40-298"><a href="de-analysis.html#cb40-298" aria-hidden="true" tabindex="-1"></a><span class="do">## ./Outputs/ Directory exists</span></span>
<span id="cb40-299"><a href="de-analysis.html#cb40-299" aria-hidden="true" tabindex="-1"></a><span class="do">## Normalised tables exported to the sub-directory: ./Outputs/</span></span>
<span id="cb40-300"><a href="de-analysis.html#cb40-300" aria-hidden="true" tabindex="-1"></a><span class="do">## PIF calculations - Preparing normalised data...</span></span>
<span id="cb40-301"><a href="de-analysis.html#cb40-301" aria-hidden="true" tabindex="-1"></a><span class="do">## PIF calculations - Preparing normalised data...COMPLETE</span></span>
<span id="cb40-302"><a href="de-analysis.html#cb40-302" aria-hidden="true" tabindex="-1"></a><span class="do">## PIF calculations - Calculating PIF data...</span></span>
<span id="cb40-303"><a href="de-analysis.html#cb40-303" aria-hidden="true" tabindex="-1"></a><span class="do">## PIF_LRvsMAL</span></span>
<span id="cb40-304"><a href="de-analysis.html#cb40-304" aria-hidden="true" tabindex="-1"></a><span class="do">## PIF_LRvsLAL</span></span>
<span id="cb40-305"><a href="de-analysis.html#cb40-305" aria-hidden="true" tabindex="-1"></a><span class="do">## PIF_MALvsLAL</span></span>
<span id="cb40-306"><a href="de-analysis.html#cb40-306" aria-hidden="true" tabindex="-1"></a><span class="do">## PIF calculations - Calculating PIF data...COMPLETE</span></span>
<span id="cb40-307"><a href="de-analysis.html#cb40-307" aria-hidden="true" tabindex="-1"></a><span class="do">## ./Outputs/ Directory exists</span></span>
<span id="cb40-308"><a href="de-analysis.html#cb40-308" aria-hidden="true" tabindex="-1"></a><span class="do">## PIF tables exported to an .xlsx file in the sub-directory: ./Outputs/</span></span>
<span id="cb40-309"><a href="de-analysis.html#cb40-309" aria-hidden="true" tabindex="-1"></a><span class="do">## PIF calculations - </span><span class="re">END</span></span>
<span id="cb40-310"><a href="de-analysis.html#cb40-310" aria-hidden="true" tabindex="-1"></a><span class="do">## ./Outputs/ Directory exists</span></span>
<span id="cb40-311"><a href="de-analysis.html#cb40-311" aria-hidden="true" tabindex="-1"></a><span class="do">## DE tables exported to an .xlsx file in the sub-directory: ./Outputs/</span></span>
<span id="cb40-312"><a href="de-analysis.html#cb40-312" aria-hidden="true" tabindex="-1"></a><span class="do">## Preparing data for output...</span></span>
<span id="cb40-313"><a href="de-analysis.html#cb40-313" aria-hidden="true" tabindex="-1"></a><span class="do">## List output succesfully generated.</span></span>
<span id="cb40-314"><a href="de-analysis.html#cb40-314" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-315"><a href="de-analysis.html#cb40-315" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-316"><a href="de-analysis.html#cb40-316" aria-hidden="true" tabindex="-1"></a><span class="do">##  ******************* </span><span class="re">END</span><span class="do"> of -  ABO *******************</span></span>
<span id="cb40-317"><a href="de-analysis.html#cb40-317" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-318"><a href="de-analysis.html#cb40-318" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-319"><a href="de-analysis.html#cb40-319" aria-hidden="true" tabindex="-1"></a><span class="do">##  ******************* Start of -  DUO *******************</span></span>
<span id="cb40-320"><a href="de-analysis.html#cb40-320" aria-hidden="true" tabindex="-1"></a><span class="do">## renaming the first element in assays to &#39;counts&#39;</span></span>
<span id="cb40-321"><a href="de-analysis.html#cb40-321" aria-hidden="true" tabindex="-1"></a><span class="do">## Warning in DESeq2::DESeqDataSet(se_data0, design = DESeq2_formula_design): some</span></span>
<span id="cb40-322"><a href="de-analysis.html#cb40-322" aria-hidden="true" tabindex="-1"></a><span class="do">## variables in design formula are characters, converting to factors</span></span>
<span id="cb40-323"><a href="de-analysis.html#cb40-323" aria-hidden="true" tabindex="-1"></a><span class="do">## Beginning DESeq analysis...</span></span>
<span id="cb40-324"><a href="de-analysis.html#cb40-324" aria-hidden="true" tabindex="-1"></a><span class="do">## estimating size factors</span></span>
<span id="cb40-325"><a href="de-analysis.html#cb40-325" aria-hidden="true" tabindex="-1"></a><span class="do">## estimating dispersions</span></span>
<span id="cb40-326"><a href="de-analysis.html#cb40-326" aria-hidden="true" tabindex="-1"></a><span class="do">## gene-wise dispersion estimates</span></span>
<span id="cb40-327"><a href="de-analysis.html#cb40-327" aria-hidden="true" tabindex="-1"></a><span class="do">## mean-dispersion relationship</span></span>
<span id="cb40-328"><a href="de-analysis.html#cb40-328" aria-hidden="true" tabindex="-1"></a><span class="do">## final dispersion estimates</span></span>
<span id="cb40-329"><a href="de-analysis.html#cb40-329" aria-hidden="true" tabindex="-1"></a><span class="do">## fitting model and testing</span></span>
<span id="cb40-330"><a href="de-analysis.html#cb40-330" aria-hidden="true" tabindex="-1"></a><span class="do">## Completed DESeq analysis.</span></span>
<span id="cb40-331"><a href="de-analysis.html#cb40-331" aria-hidden="true" tabindex="-1"></a><span class="do">## Plotting cooks distance...</span></span>
<span id="cb40-332"><a href="de-analysis.html#cb40-332" aria-hidden="true" tabindex="-1"></a><span class="do">## Cooks distance plot complete.</span></span>
<span id="cb40-333"><a href="de-analysis.html#cb40-333" aria-hidden="true" tabindex="-1"></a><span class="do">## Generating pairwise DESeq2 results...</span></span>
<span id="cb40-334"><a href="de-analysis.html#cb40-334" aria-hidden="true" tabindex="-1"></a><span class="do">## [1] &quot;Contrast levels: LR, MAL, LAL&quot;</span></span>
<span id="cb40-335"><a href="de-analysis.html#cb40-335" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-336"><a href="de-analysis.html#cb40-336" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-337"><a href="de-analysis.html#cb40-337" aria-hidden="true" tabindex="-1"></a><span class="do">##  log2 fold change (MLE): Treatment LR vs MAL</span></span>
<span id="cb40-338"><a href="de-analysis.html#cb40-338" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-339"><a href="de-analysis.html#cb40-339" aria-hidden="true" tabindex="-1"></a><span class="do">## out of 18958 with nonzero total read count</span></span>
<span id="cb40-340"><a href="de-analysis.html#cb40-340" aria-hidden="true" tabindex="-1"></a><span class="do">## adjusted p-value &lt; 0.05</span></span>
<span id="cb40-341"><a href="de-analysis.html#cb40-341" aria-hidden="true" tabindex="-1"></a><span class="do">## LFC &gt; 0 (up)       : 466, 2.5%</span></span>
<span id="cb40-342"><a href="de-analysis.html#cb40-342" aria-hidden="true" tabindex="-1"></a><span class="do">## LFC &lt; 0 (down)     : 323, 1.7%</span></span>
<span id="cb40-343"><a href="de-analysis.html#cb40-343" aria-hidden="true" tabindex="-1"></a><span class="do">## outliers [1]       : 59, 0.31%</span></span>
<span id="cb40-344"><a href="de-analysis.html#cb40-344" aria-hidden="true" tabindex="-1"></a><span class="do">## [1] see &#39;cooksCutoff&#39; argument of ?results</span></span>
<span id="cb40-345"><a href="de-analysis.html#cb40-345" aria-hidden="true" tabindex="-1"></a><span class="do">## see metadata(res)$ihwResult on hypothesis weighting</span></span>
<span id="cb40-346"><a href="de-analysis.html#cb40-346" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-347"><a href="de-analysis.html#cb40-347" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-348"><a href="de-analysis.html#cb40-348" aria-hidden="true" tabindex="-1"></a><span class="do">##  log2 fold change (MLE): Treatment LR vs LAL</span></span>
<span id="cb40-349"><a href="de-analysis.html#cb40-349" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-350"><a href="de-analysis.html#cb40-350" aria-hidden="true" tabindex="-1"></a><span class="do">## out of 18958 with nonzero total read count</span></span>
<span id="cb40-351"><a href="de-analysis.html#cb40-351" aria-hidden="true" tabindex="-1"></a><span class="do">## adjusted p-value &lt; 0.05</span></span>
<span id="cb40-352"><a href="de-analysis.html#cb40-352" aria-hidden="true" tabindex="-1"></a><span class="do">## LFC &gt; 0 (up)       : 36, 0.19%</span></span>
<span id="cb40-353"><a href="de-analysis.html#cb40-353" aria-hidden="true" tabindex="-1"></a><span class="do">## LFC &lt; 0 (down)     : 39, 0.21%</span></span>
<span id="cb40-354"><a href="de-analysis.html#cb40-354" aria-hidden="true" tabindex="-1"></a><span class="do">## outliers [1]       : 59, 0.31%</span></span>
<span id="cb40-355"><a href="de-analysis.html#cb40-355" aria-hidden="true" tabindex="-1"></a><span class="do">## [1] see &#39;cooksCutoff&#39; argument of ?results</span></span>
<span id="cb40-356"><a href="de-analysis.html#cb40-356" aria-hidden="true" tabindex="-1"></a><span class="do">## see metadata(res)$ihwResult on hypothesis weighting</span></span>
<span id="cb40-357"><a href="de-analysis.html#cb40-357" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-358"><a href="de-analysis.html#cb40-358" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-359"><a href="de-analysis.html#cb40-359" aria-hidden="true" tabindex="-1"></a><span class="do">##  log2 fold change (MLE): Treatment MAL vs LAL</span></span>
<span id="cb40-360"><a href="de-analysis.html#cb40-360" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-361"><a href="de-analysis.html#cb40-361" aria-hidden="true" tabindex="-1"></a><span class="do">## out of 18958 with nonzero total read count</span></span>
<span id="cb40-362"><a href="de-analysis.html#cb40-362" aria-hidden="true" tabindex="-1"></a><span class="do">## adjusted p-value &lt; 0.05</span></span>
<span id="cb40-363"><a href="de-analysis.html#cb40-363" aria-hidden="true" tabindex="-1"></a><span class="do">## LFC &gt; 0 (up)       : 64, 0.34%</span></span>
<span id="cb40-364"><a href="de-analysis.html#cb40-364" aria-hidden="true" tabindex="-1"></a><span class="do">## LFC &lt; 0 (down)     : 146, 0.77%</span></span>
<span id="cb40-365"><a href="de-analysis.html#cb40-365" aria-hidden="true" tabindex="-1"></a><span class="do">## outliers [1]       : 59, 0.31%</span></span>
<span id="cb40-366"><a href="de-analysis.html#cb40-366" aria-hidden="true" tabindex="-1"></a><span class="do">## [1] see &#39;cooksCutoff&#39; argument of ?results</span></span>
<span id="cb40-367"><a href="de-analysis.html#cb40-367" aria-hidden="true" tabindex="-1"></a><span class="do">## see metadata(res)$ihwResult on hypothesis weighting</span></span>
<span id="cb40-368"><a href="de-analysis.html#cb40-368" aria-hidden="true" tabindex="-1"></a><span class="do">## Results generated.</span></span>
<span id="cb40-369"><a href="de-analysis.html#cb40-369" aria-hidden="true" tabindex="-1"></a><span class="do">## Plotting MA plots (default DESeq2 style for QC)...</span></span>
<span id="cb40-370"><a href="de-analysis.html#cb40-370" aria-hidden="true" tabindex="-1"></a><span class="do">## Finished MA plots.</span></span>
<span id="cb40-371"><a href="de-analysis.html#cb40-371" aria-hidden="true" tabindex="-1"></a><span class="do">## Plotting p value histograms (for QC)...</span></span>
<span id="cb40-372"><a href="de-analysis.html#cb40-372" aria-hidden="true" tabindex="-1"></a><span class="do">## Finished plotting p value histograms.</span></span>
<span id="cb40-373"><a href="de-analysis.html#cb40-373" aria-hidden="true" tabindex="-1"></a><span class="do">## ./Outputs/ Directory exists</span></span>
<span id="cb40-374"><a href="de-analysis.html#cb40-374" aria-hidden="true" tabindex="-1"></a><span class="do">## Normalised tables exported to the sub-directory: ./Outputs/</span></span>
<span id="cb40-375"><a href="de-analysis.html#cb40-375" aria-hidden="true" tabindex="-1"></a><span class="do">## PIF calculations - Preparing normalised data...</span></span>
<span id="cb40-376"><a href="de-analysis.html#cb40-376" aria-hidden="true" tabindex="-1"></a><span class="do">## PIF calculations - Preparing normalised data...COMPLETE</span></span>
<span id="cb40-377"><a href="de-analysis.html#cb40-377" aria-hidden="true" tabindex="-1"></a><span class="do">## PIF calculations - Calculating PIF data...</span></span>
<span id="cb40-378"><a href="de-analysis.html#cb40-378" aria-hidden="true" tabindex="-1"></a><span class="do">## PIF_LRvsMAL</span></span>
<span id="cb40-379"><a href="de-analysis.html#cb40-379" aria-hidden="true" tabindex="-1"></a><span class="do">## PIF_LRvsLAL</span></span>
<span id="cb40-380"><a href="de-analysis.html#cb40-380" aria-hidden="true" tabindex="-1"></a><span class="do">## PIF_MALvsLAL</span></span>
<span id="cb40-381"><a href="de-analysis.html#cb40-381" aria-hidden="true" tabindex="-1"></a><span class="do">## PIF calculations - Calculating PIF data...COMPLETE</span></span>
<span id="cb40-382"><a href="de-analysis.html#cb40-382" aria-hidden="true" tabindex="-1"></a><span class="do">## ./Outputs/ Directory exists</span></span>
<span id="cb40-383"><a href="de-analysis.html#cb40-383" aria-hidden="true" tabindex="-1"></a><span class="do">## PIF tables exported to an .xlsx file in the sub-directory: ./Outputs/</span></span>
<span id="cb40-384"><a href="de-analysis.html#cb40-384" aria-hidden="true" tabindex="-1"></a><span class="do">## PIF calculations - </span><span class="re">END</span></span>
<span id="cb40-385"><a href="de-analysis.html#cb40-385" aria-hidden="true" tabindex="-1"></a><span class="do">## ./Outputs/ Directory exists</span></span>
<span id="cb40-386"><a href="de-analysis.html#cb40-386" aria-hidden="true" tabindex="-1"></a><span class="do">## DE tables exported to an .xlsx file in the sub-directory: ./Outputs/</span></span>
<span id="cb40-387"><a href="de-analysis.html#cb40-387" aria-hidden="true" tabindex="-1"></a><span class="do">## Preparing data for output...</span></span>
<span id="cb40-388"><a href="de-analysis.html#cb40-388" aria-hidden="true" tabindex="-1"></a><span class="do">## List output succesfully generated.</span></span>
<span id="cb40-389"><a href="de-analysis.html#cb40-389" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-390"><a href="de-analysis.html#cb40-390" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-391"><a href="de-analysis.html#cb40-391" aria-hidden="true" tabindex="-1"></a><span class="do">##  ******************* </span><span class="re">END</span><span class="do"> of -  DUO *******************</span></span>
<span id="cb40-392"><a href="de-analysis.html#cb40-392" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-393"><a href="de-analysis.html#cb40-393" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-394"><a href="de-analysis.html#cb40-394" aria-hidden="true" tabindex="-1"></a><span class="do">##  ******************* Start of -  RUM *******************</span></span>
<span id="cb40-395"><a href="de-analysis.html#cb40-395" aria-hidden="true" tabindex="-1"></a><span class="do">## renaming the first element in assays to &#39;counts&#39;</span></span>
<span id="cb40-396"><a href="de-analysis.html#cb40-396" aria-hidden="true" tabindex="-1"></a><span class="do">## Warning in DESeq2::DESeqDataSet(se_data0, design = DESeq2_formula_design): some</span></span>
<span id="cb40-397"><a href="de-analysis.html#cb40-397" aria-hidden="true" tabindex="-1"></a><span class="do">## variables in design formula are characters, converting to factors</span></span>
<span id="cb40-398"><a href="de-analysis.html#cb40-398" aria-hidden="true" tabindex="-1"></a><span class="do">## Beginning DESeq analysis...</span></span>
<span id="cb40-399"><a href="de-analysis.html#cb40-399" aria-hidden="true" tabindex="-1"></a><span class="do">## estimating size factors</span></span>
<span id="cb40-400"><a href="de-analysis.html#cb40-400" aria-hidden="true" tabindex="-1"></a><span class="do">## estimating dispersions</span></span>
<span id="cb40-401"><a href="de-analysis.html#cb40-401" aria-hidden="true" tabindex="-1"></a><span class="do">## gene-wise dispersion estimates</span></span>
<span id="cb40-402"><a href="de-analysis.html#cb40-402" aria-hidden="true" tabindex="-1"></a><span class="do">## mean-dispersion relationship</span></span>
<span id="cb40-403"><a href="de-analysis.html#cb40-403" aria-hidden="true" tabindex="-1"></a><span class="do">## final dispersion estimates</span></span>
<span id="cb40-404"><a href="de-analysis.html#cb40-404" aria-hidden="true" tabindex="-1"></a><span class="do">## fitting model and testing</span></span>
<span id="cb40-405"><a href="de-analysis.html#cb40-405" aria-hidden="true" tabindex="-1"></a><span class="do">## Completed DESeq analysis.</span></span>
<span id="cb40-406"><a href="de-analysis.html#cb40-406" aria-hidden="true" tabindex="-1"></a><span class="do">## Plotting cooks distance...</span></span>
<span id="cb40-407"><a href="de-analysis.html#cb40-407" aria-hidden="true" tabindex="-1"></a><span class="do">## Cooks distance plot complete.</span></span>
<span id="cb40-408"><a href="de-analysis.html#cb40-408" aria-hidden="true" tabindex="-1"></a><span class="do">## Generating pairwise DESeq2 results...</span></span>
<span id="cb40-409"><a href="de-analysis.html#cb40-409" aria-hidden="true" tabindex="-1"></a><span class="do">## [1] &quot;Contrast levels: LR, MAL, LAL&quot;</span></span>
<span id="cb40-410"><a href="de-analysis.html#cb40-410" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-411"><a href="de-analysis.html#cb40-411" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-412"><a href="de-analysis.html#cb40-412" aria-hidden="true" tabindex="-1"></a><span class="do">##  log2 fold change (MLE): Treatment LR vs MAL</span></span>
<span id="cb40-413"><a href="de-analysis.html#cb40-413" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-414"><a href="de-analysis.html#cb40-414" aria-hidden="true" tabindex="-1"></a><span class="do">## out of 17655 with nonzero total read count</span></span>
<span id="cb40-415"><a href="de-analysis.html#cb40-415" aria-hidden="true" tabindex="-1"></a><span class="do">## adjusted p-value &lt; 0.05</span></span>
<span id="cb40-416"><a href="de-analysis.html#cb40-416" aria-hidden="true" tabindex="-1"></a><span class="do">## LFC &gt; 0 (up)       : 207, 1.2%</span></span>
<span id="cb40-417"><a href="de-analysis.html#cb40-417" aria-hidden="true" tabindex="-1"></a><span class="do">## LFC &lt; 0 (down)     : 280, 1.6%</span></span>
<span id="cb40-418"><a href="de-analysis.html#cb40-418" aria-hidden="true" tabindex="-1"></a><span class="do">## outliers [1]       : 139, 0.79%</span></span>
<span id="cb40-419"><a href="de-analysis.html#cb40-419" aria-hidden="true" tabindex="-1"></a><span class="do">## [1] see &#39;cooksCutoff&#39; argument of ?results</span></span>
<span id="cb40-420"><a href="de-analysis.html#cb40-420" aria-hidden="true" tabindex="-1"></a><span class="do">## see metadata(res)$ihwResult on hypothesis weighting</span></span>
<span id="cb40-421"><a href="de-analysis.html#cb40-421" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-422"><a href="de-analysis.html#cb40-422" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-423"><a href="de-analysis.html#cb40-423" aria-hidden="true" tabindex="-1"></a><span class="do">##  log2 fold change (MLE): Treatment LR vs LAL</span></span>
<span id="cb40-424"><a href="de-analysis.html#cb40-424" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-425"><a href="de-analysis.html#cb40-425" aria-hidden="true" tabindex="-1"></a><span class="do">## out of 17655 with nonzero total read count</span></span>
<span id="cb40-426"><a href="de-analysis.html#cb40-426" aria-hidden="true" tabindex="-1"></a><span class="do">## adjusted p-value &lt; 0.05</span></span>
<span id="cb40-427"><a href="de-analysis.html#cb40-427" aria-hidden="true" tabindex="-1"></a><span class="do">## LFC &gt; 0 (up)       : 25, 0.14%</span></span>
<span id="cb40-428"><a href="de-analysis.html#cb40-428" aria-hidden="true" tabindex="-1"></a><span class="do">## LFC &lt; 0 (down)     : 18, 0.1%</span></span>
<span id="cb40-429"><a href="de-analysis.html#cb40-429" aria-hidden="true" tabindex="-1"></a><span class="do">## outliers [1]       : 139, 0.79%</span></span>
<span id="cb40-430"><a href="de-analysis.html#cb40-430" aria-hidden="true" tabindex="-1"></a><span class="do">## [1] see &#39;cooksCutoff&#39; argument of ?results</span></span>
<span id="cb40-431"><a href="de-analysis.html#cb40-431" aria-hidden="true" tabindex="-1"></a><span class="do">## see metadata(res)$ihwResult on hypothesis weighting</span></span>
<span id="cb40-432"><a href="de-analysis.html#cb40-432" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-433"><a href="de-analysis.html#cb40-433" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-434"><a href="de-analysis.html#cb40-434" aria-hidden="true" tabindex="-1"></a><span class="do">##  log2 fold change (MLE): Treatment MAL vs LAL</span></span>
<span id="cb40-435"><a href="de-analysis.html#cb40-435" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-436"><a href="de-analysis.html#cb40-436" aria-hidden="true" tabindex="-1"></a><span class="do">## out of 17655 with nonzero total read count</span></span>
<span id="cb40-437"><a href="de-analysis.html#cb40-437" aria-hidden="true" tabindex="-1"></a><span class="do">## adjusted p-value &lt; 0.05</span></span>
<span id="cb40-438"><a href="de-analysis.html#cb40-438" aria-hidden="true" tabindex="-1"></a><span class="do">## LFC &gt; 0 (up)       : 155, 0.88%</span></span>
<span id="cb40-439"><a href="de-analysis.html#cb40-439" aria-hidden="true" tabindex="-1"></a><span class="do">## LFC &lt; 0 (down)     : 205, 1.2%</span></span>
<span id="cb40-440"><a href="de-analysis.html#cb40-440" aria-hidden="true" tabindex="-1"></a><span class="do">## outliers [1]       : 139, 0.79%</span></span>
<span id="cb40-441"><a href="de-analysis.html#cb40-441" aria-hidden="true" tabindex="-1"></a><span class="do">## [1] see &#39;cooksCutoff&#39; argument of ?results</span></span>
<span id="cb40-442"><a href="de-analysis.html#cb40-442" aria-hidden="true" tabindex="-1"></a><span class="do">## see metadata(res)$ihwResult on hypothesis weighting</span></span>
<span id="cb40-443"><a href="de-analysis.html#cb40-443" aria-hidden="true" tabindex="-1"></a><span class="do">## Results generated.</span></span>
<span id="cb40-444"><a href="de-analysis.html#cb40-444" aria-hidden="true" tabindex="-1"></a><span class="do">## Plotting MA plots (default DESeq2 style for QC)...</span></span>
<span id="cb40-445"><a href="de-analysis.html#cb40-445" aria-hidden="true" tabindex="-1"></a><span class="do">## Finished MA plots.</span></span>
<span id="cb40-446"><a href="de-analysis.html#cb40-446" aria-hidden="true" tabindex="-1"></a><span class="do">## Plotting p value histograms (for QC)...</span></span>
<span id="cb40-447"><a href="de-analysis.html#cb40-447" aria-hidden="true" tabindex="-1"></a><span class="do">## Finished plotting p value histograms.</span></span>
<span id="cb40-448"><a href="de-analysis.html#cb40-448" aria-hidden="true" tabindex="-1"></a><span class="do">## ./Outputs/ Directory exists</span></span>
<span id="cb40-449"><a href="de-analysis.html#cb40-449" aria-hidden="true" tabindex="-1"></a><span class="do">## Normalised tables exported to the sub-directory: ./Outputs/</span></span>
<span id="cb40-450"><a href="de-analysis.html#cb40-450" aria-hidden="true" tabindex="-1"></a><span class="do">## PIF calculations - Preparing normalised data...</span></span>
<span id="cb40-451"><a href="de-analysis.html#cb40-451" aria-hidden="true" tabindex="-1"></a><span class="do">## PIF calculations - Preparing normalised data...COMPLETE</span></span>
<span id="cb40-452"><a href="de-analysis.html#cb40-452" aria-hidden="true" tabindex="-1"></a><span class="do">## PIF calculations - Calculating PIF data...</span></span>
<span id="cb40-453"><a href="de-analysis.html#cb40-453" aria-hidden="true" tabindex="-1"></a><span class="do">## PIF_LRvsMAL</span></span>
<span id="cb40-454"><a href="de-analysis.html#cb40-454" aria-hidden="true" tabindex="-1"></a><span class="do">## PIF_LRvsLAL</span></span>
<span id="cb40-455"><a href="de-analysis.html#cb40-455" aria-hidden="true" tabindex="-1"></a><span class="do">## PIF_MALvsLAL</span></span>
<span id="cb40-456"><a href="de-analysis.html#cb40-456" aria-hidden="true" tabindex="-1"></a><span class="do">## PIF calculations - Calculating PIF data...COMPLETE</span></span>
<span id="cb40-457"><a href="de-analysis.html#cb40-457" aria-hidden="true" tabindex="-1"></a><span class="do">## ./Outputs/ Directory exists</span></span>
<span id="cb40-458"><a href="de-analysis.html#cb40-458" aria-hidden="true" tabindex="-1"></a><span class="do">## PIF tables exported to an .xlsx file in the sub-directory: ./Outputs/</span></span>
<span id="cb40-459"><a href="de-analysis.html#cb40-459" aria-hidden="true" tabindex="-1"></a><span class="do">## PIF calculations - </span><span class="re">END</span></span>
<span id="cb40-460"><a href="de-analysis.html#cb40-460" aria-hidden="true" tabindex="-1"></a><span class="do">## ./Outputs/ Directory exists</span></span>
<span id="cb40-461"><a href="de-analysis.html#cb40-461" aria-hidden="true" tabindex="-1"></a><span class="do">## DE tables exported to an .xlsx file in the sub-directory: ./Outputs/</span></span>
<span id="cb40-462"><a href="de-analysis.html#cb40-462" aria-hidden="true" tabindex="-1"></a><span class="do">## Preparing data for output...</span></span>
<span id="cb40-463"><a href="de-analysis.html#cb40-463" aria-hidden="true" tabindex="-1"></a><span class="do">## List output succesfully generated.</span></span>
<span id="cb40-464"><a href="de-analysis.html#cb40-464" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-465"><a href="de-analysis.html#cb40-465" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-466"><a href="de-analysis.html#cb40-466" aria-hidden="true" tabindex="-1"></a><span class="do">##  ******************* </span><span class="re">END</span><span class="do"> of -  RUM *******************</span></span>
<span id="cb40-467"><a href="de-analysis.html#cb40-467" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-468"><a href="de-analysis.html#cb40-468" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-469"><a href="de-analysis.html#cb40-469" aria-hidden="true" tabindex="-1"></a><span class="do">##  ******************* Start of -  LIV *******************</span></span>
<span id="cb40-470"><a href="de-analysis.html#cb40-470" aria-hidden="true" tabindex="-1"></a><span class="do">## renaming the first element in assays to &#39;counts&#39;</span></span>
<span id="cb40-471"><a href="de-analysis.html#cb40-471" aria-hidden="true" tabindex="-1"></a><span class="do">## Warning in DESeq2::DESeqDataSet(se_data0, design = DESeq2_formula_design): some</span></span>
<span id="cb40-472"><a href="de-analysis.html#cb40-472" aria-hidden="true" tabindex="-1"></a><span class="do">## variables in design formula are characters, converting to factors</span></span>
<span id="cb40-473"><a href="de-analysis.html#cb40-473" aria-hidden="true" tabindex="-1"></a><span class="do">## Beginning DESeq analysis...</span></span>
<span id="cb40-474"><a href="de-analysis.html#cb40-474" aria-hidden="true" tabindex="-1"></a><span class="do">## estimating size factors</span></span>
<span id="cb40-475"><a href="de-analysis.html#cb40-475" aria-hidden="true" tabindex="-1"></a><span class="do">## estimating dispersions</span></span>
<span id="cb40-476"><a href="de-analysis.html#cb40-476" aria-hidden="true" tabindex="-1"></a><span class="do">## gene-wise dispersion estimates</span></span>
<span id="cb40-477"><a href="de-analysis.html#cb40-477" aria-hidden="true" tabindex="-1"></a><span class="do">## mean-dispersion relationship</span></span>
<span id="cb40-478"><a href="de-analysis.html#cb40-478" aria-hidden="true" tabindex="-1"></a><span class="do">## final dispersion estimates</span></span>
<span id="cb40-479"><a href="de-analysis.html#cb40-479" aria-hidden="true" tabindex="-1"></a><span class="do">## fitting model and testing</span></span>
<span id="cb40-480"><a href="de-analysis.html#cb40-480" aria-hidden="true" tabindex="-1"></a><span class="do">## Completed DESeq analysis.</span></span>
<span id="cb40-481"><a href="de-analysis.html#cb40-481" aria-hidden="true" tabindex="-1"></a><span class="do">## Plotting cooks distance...</span></span>
<span id="cb40-482"><a href="de-analysis.html#cb40-482" aria-hidden="true" tabindex="-1"></a><span class="do">## Cooks distance plot complete.</span></span>
<span id="cb40-483"><a href="de-analysis.html#cb40-483" aria-hidden="true" tabindex="-1"></a><span class="do">## Generating pairwise DESeq2 results...</span></span>
<span id="cb40-484"><a href="de-analysis.html#cb40-484" aria-hidden="true" tabindex="-1"></a><span class="do">## [1] &quot;Contrast levels: LR, MAL, LAL&quot;</span></span>
<span id="cb40-485"><a href="de-analysis.html#cb40-485" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-486"><a href="de-analysis.html#cb40-486" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-487"><a href="de-analysis.html#cb40-487" aria-hidden="true" tabindex="-1"></a><span class="do">##  log2 fold change (MLE): Treatment LR vs MAL</span></span>
<span id="cb40-488"><a href="de-analysis.html#cb40-488" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-489"><a href="de-analysis.html#cb40-489" aria-hidden="true" tabindex="-1"></a><span class="do">## out of 17765 with nonzero total read count</span></span>
<span id="cb40-490"><a href="de-analysis.html#cb40-490" aria-hidden="true" tabindex="-1"></a><span class="do">## adjusted p-value &lt; 0.05</span></span>
<span id="cb40-491"><a href="de-analysis.html#cb40-491" aria-hidden="true" tabindex="-1"></a><span class="do">## LFC &gt; 0 (up)       : 28, 0.16%</span></span>
<span id="cb40-492"><a href="de-analysis.html#cb40-492" aria-hidden="true" tabindex="-1"></a><span class="do">## LFC &lt; 0 (down)     : 34, 0.19%</span></span>
<span id="cb40-493"><a href="de-analysis.html#cb40-493" aria-hidden="true" tabindex="-1"></a><span class="do">## outliers [1]       : 43, 0.24%</span></span>
<span id="cb40-494"><a href="de-analysis.html#cb40-494" aria-hidden="true" tabindex="-1"></a><span class="do">## [1] see &#39;cooksCutoff&#39; argument of ?results</span></span>
<span id="cb40-495"><a href="de-analysis.html#cb40-495" aria-hidden="true" tabindex="-1"></a><span class="do">## see metadata(res)$ihwResult on hypothesis weighting</span></span>
<span id="cb40-496"><a href="de-analysis.html#cb40-496" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-497"><a href="de-analysis.html#cb40-497" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-498"><a href="de-analysis.html#cb40-498" aria-hidden="true" tabindex="-1"></a><span class="do">##  log2 fold change (MLE): Treatment LR vs LAL</span></span>
<span id="cb40-499"><a href="de-analysis.html#cb40-499" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-500"><a href="de-analysis.html#cb40-500" aria-hidden="true" tabindex="-1"></a><span class="do">## out of 17765 with nonzero total read count</span></span>
<span id="cb40-501"><a href="de-analysis.html#cb40-501" aria-hidden="true" tabindex="-1"></a><span class="do">## adjusted p-value &lt; 0.05</span></span>
<span id="cb40-502"><a href="de-analysis.html#cb40-502" aria-hidden="true" tabindex="-1"></a><span class="do">## LFC &gt; 0 (up)       : 99, 0.56%</span></span>
<span id="cb40-503"><a href="de-analysis.html#cb40-503" aria-hidden="true" tabindex="-1"></a><span class="do">## LFC &lt; 0 (down)     : 189, 1.1%</span></span>
<span id="cb40-504"><a href="de-analysis.html#cb40-504" aria-hidden="true" tabindex="-1"></a><span class="do">## outliers [1]       : 43, 0.24%</span></span>
<span id="cb40-505"><a href="de-analysis.html#cb40-505" aria-hidden="true" tabindex="-1"></a><span class="do">## [1] see &#39;cooksCutoff&#39; argument of ?results</span></span>
<span id="cb40-506"><a href="de-analysis.html#cb40-506" aria-hidden="true" tabindex="-1"></a><span class="do">## see metadata(res)$ihwResult on hypothesis weighting</span></span>
<span id="cb40-507"><a href="de-analysis.html#cb40-507" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-508"><a href="de-analysis.html#cb40-508" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-509"><a href="de-analysis.html#cb40-509" aria-hidden="true" tabindex="-1"></a><span class="do">##  log2 fold change (MLE): Treatment MAL vs LAL</span></span>
<span id="cb40-510"><a href="de-analysis.html#cb40-510" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-511"><a href="de-analysis.html#cb40-511" aria-hidden="true" tabindex="-1"></a><span class="do">## out of 17765 with nonzero total read count</span></span>
<span id="cb40-512"><a href="de-analysis.html#cb40-512" aria-hidden="true" tabindex="-1"></a><span class="do">## adjusted p-value &lt; 0.05</span></span>
<span id="cb40-513"><a href="de-analysis.html#cb40-513" aria-hidden="true" tabindex="-1"></a><span class="do">## LFC &gt; 0 (up)       : 248, 1.4%</span></span>
<span id="cb40-514"><a href="de-analysis.html#cb40-514" aria-hidden="true" tabindex="-1"></a><span class="do">## LFC &lt; 0 (down)     : 468, 2.6%</span></span>
<span id="cb40-515"><a href="de-analysis.html#cb40-515" aria-hidden="true" tabindex="-1"></a><span class="do">## outliers [1]       : 43, 0.24%</span></span>
<span id="cb40-516"><a href="de-analysis.html#cb40-516" aria-hidden="true" tabindex="-1"></a><span class="do">## [1] see &#39;cooksCutoff&#39; argument of ?results</span></span>
<span id="cb40-517"><a href="de-analysis.html#cb40-517" aria-hidden="true" tabindex="-1"></a><span class="do">## see metadata(res)$ihwResult on hypothesis weighting</span></span>
<span id="cb40-518"><a href="de-analysis.html#cb40-518" aria-hidden="true" tabindex="-1"></a><span class="do">## Results generated.</span></span>
<span id="cb40-519"><a href="de-analysis.html#cb40-519" aria-hidden="true" tabindex="-1"></a><span class="do">## Plotting MA plots (default DESeq2 style for QC)...</span></span>
<span id="cb40-520"><a href="de-analysis.html#cb40-520" aria-hidden="true" tabindex="-1"></a><span class="do">## Finished MA plots.</span></span>
<span id="cb40-521"><a href="de-analysis.html#cb40-521" aria-hidden="true" tabindex="-1"></a><span class="do">## Plotting p value histograms (for QC)...</span></span>
<span id="cb40-522"><a href="de-analysis.html#cb40-522" aria-hidden="true" tabindex="-1"></a><span class="do">## Finished plotting p value histograms.</span></span>
<span id="cb40-523"><a href="de-analysis.html#cb40-523" aria-hidden="true" tabindex="-1"></a><span class="do">## ./Outputs/ Directory exists</span></span>
<span id="cb40-524"><a href="de-analysis.html#cb40-524" aria-hidden="true" tabindex="-1"></a><span class="do">## Normalised tables exported to the sub-directory: ./Outputs/</span></span>
<span id="cb40-525"><a href="de-analysis.html#cb40-525" aria-hidden="true" tabindex="-1"></a><span class="do">## PIF calculations - Preparing normalised data...</span></span>
<span id="cb40-526"><a href="de-analysis.html#cb40-526" aria-hidden="true" tabindex="-1"></a><span class="do">## PIF calculations - Preparing normalised data...COMPLETE</span></span>
<span id="cb40-527"><a href="de-analysis.html#cb40-527" aria-hidden="true" tabindex="-1"></a><span class="do">## PIF calculations - Calculating PIF data...</span></span>
<span id="cb40-528"><a href="de-analysis.html#cb40-528" aria-hidden="true" tabindex="-1"></a><span class="do">## PIF_LRvsMAL</span></span>
<span id="cb40-529"><a href="de-analysis.html#cb40-529" aria-hidden="true" tabindex="-1"></a><span class="do">## PIF_LRvsLAL</span></span>
<span id="cb40-530"><a href="de-analysis.html#cb40-530" aria-hidden="true" tabindex="-1"></a><span class="do">## PIF_MALvsLAL</span></span>
<span id="cb40-531"><a href="de-analysis.html#cb40-531" aria-hidden="true" tabindex="-1"></a><span class="do">## PIF calculations - Calculating PIF data...COMPLETE</span></span>
<span id="cb40-532"><a href="de-analysis.html#cb40-532" aria-hidden="true" tabindex="-1"></a><span class="do">## ./Outputs/ Directory exists</span></span>
<span id="cb40-533"><a href="de-analysis.html#cb40-533" aria-hidden="true" tabindex="-1"></a><span class="do">## PIF tables exported to an .xlsx file in the sub-directory: ./Outputs/</span></span>
<span id="cb40-534"><a href="de-analysis.html#cb40-534" aria-hidden="true" tabindex="-1"></a><span class="do">## PIF calculations - </span><span class="re">END</span></span>
<span id="cb40-535"><a href="de-analysis.html#cb40-535" aria-hidden="true" tabindex="-1"></a><span class="do">## ./Outputs/ Directory exists</span></span>
<span id="cb40-536"><a href="de-analysis.html#cb40-536" aria-hidden="true" tabindex="-1"></a><span class="do">## DE tables exported to an .xlsx file in the sub-directory: ./Outputs/</span></span>
<span id="cb40-537"><a href="de-analysis.html#cb40-537" aria-hidden="true" tabindex="-1"></a><span class="do">## Preparing data for output...</span></span>
<span id="cb40-538"><a href="de-analysis.html#cb40-538" aria-hidden="true" tabindex="-1"></a><span class="do">## List output succesfully generated.</span></span>
<span id="cb40-539"><a href="de-analysis.html#cb40-539" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-540"><a href="de-analysis.html#cb40-540" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb40-541"><a href="de-analysis.html#cb40-541" aria-hidden="true" tabindex="-1"></a><span class="do">##  ******************* </span><span class="re">END</span><span class="do"> of -  LIV *******************</span></span>
<span id="cb40-542"><a href="de-analysis.html#cb40-542" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb40-543"><a href="de-analysis.html#cb40-543" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb40-544"><a href="de-analysis.html#cb40-544" aria-hidden="true" tabindex="-1"></a><span class="co">#View sructure of DE_out object</span></span>
<span id="cb40-545"><a href="de-analysis.html#cb40-545" aria-hidden="true" tabindex="-1"></a><span class="fu">str</span>(DE_out, <span class="at">max.level =</span> <span class="dv">2</span>)</span>
<span id="cb40-546"><a href="de-analysis.html#cb40-546" aria-hidden="true" tabindex="-1"></a><span class="do">## List of 7</span></span>
<span id="cb40-547"><a href="de-analysis.html#cb40-547" aria-hidden="true" tabindex="-1"></a><span class="do">##  $ ARC_DESeq2_Output:List of 8</span></span>
<span id="cb40-548"><a href="de-analysis.html#cb40-548" aria-hidden="true" tabindex="-1"></a><span class="do">##   ..$ dds_wald_object       :Formal class &#39;DESeqDataSet&#39; [package &quot;DESeq2&quot;] with 8 slots</span></span>
<span id="cb40-549"><a href="de-analysis.html#cb40-549" aria-hidden="true" tabindex="-1"></a><span class="do">##   ..$ boxplot_cooks_distance:List of 9</span></span>
<span id="cb40-550"><a href="de-analysis.html#cb40-550" aria-hidden="true" tabindex="-1"></a><span class="do">##   .. ..- attr(*, &quot;class&quot;)= chr [1:2] &quot;gg&quot; &quot;ggplot&quot;</span></span>
<span id="cb40-551"><a href="de-analysis.html#cb40-551" aria-hidden="true" tabindex="-1"></a><span class="do">##   ..$ DESeq2_res_object     :List of 3</span></span>
<span id="cb40-552"><a href="de-analysis.html#cb40-552" aria-hidden="true" tabindex="-1"></a><span class="do">##   ..$ pairwise_plots        :List of 2</span></span>
<span id="cb40-553"><a href="de-analysis.html#cb40-553" aria-hidden="true" tabindex="-1"></a><span class="do">##   ..$ overall_plots         :List of 3</span></span>
<span id="cb40-554"><a href="de-analysis.html#cb40-554" aria-hidden="true" tabindex="-1"></a><span class="do">##   ..$ normalised_data       :List of 2</span></span>
<span id="cb40-555"><a href="de-analysis.html#cb40-555" aria-hidden="true" tabindex="-1"></a><span class="do">##   ..$ PIF                   :List of 3</span></span>
<span id="cb40-556"><a href="de-analysis.html#cb40-556" aria-hidden="true" tabindex="-1"></a><span class="do">##   ..$ DE_by_PIF_df          :List of 3</span></span>
<span id="cb40-557"><a href="de-analysis.html#cb40-557" aria-hidden="true" tabindex="-1"></a><span class="do">##  $ LHA_DESeq2_Output:List of 8</span></span>
<span id="cb40-558"><a href="de-analysis.html#cb40-558" aria-hidden="true" tabindex="-1"></a><span class="do">##   ..$ dds_wald_object       :Formal class &#39;DESeqDataSet&#39; [package &quot;DESeq2&quot;] with 8 slots</span></span>
<span id="cb40-559"><a href="de-analysis.html#cb40-559" aria-hidden="true" tabindex="-1"></a><span class="do">##   ..$ boxplot_cooks_distance:List of 9</span></span>
<span id="cb40-560"><a href="de-analysis.html#cb40-560" aria-hidden="true" tabindex="-1"></a><span class="do">##   .. ..- attr(*, &quot;class&quot;)= chr [1:2] &quot;gg&quot; &quot;ggplot&quot;</span></span>
<span id="cb40-561"><a href="de-analysis.html#cb40-561" aria-hidden="true" tabindex="-1"></a><span class="do">##   ..$ DESeq2_res_object     :List of 3</span></span>
<span id="cb40-562"><a href="de-analysis.html#cb40-562" aria-hidden="true" tabindex="-1"></a><span class="do">##   ..$ pairwise_plots        :List of 2</span></span>
<span id="cb40-563"><a href="de-analysis.html#cb40-563" aria-hidden="true" tabindex="-1"></a><span class="do">##   ..$ overall_plots         :List of 3</span></span>
<span id="cb40-564"><a href="de-analysis.html#cb40-564" aria-hidden="true" tabindex="-1"></a><span class="do">##   ..$ normalised_data       :List of 2</span></span>
<span id="cb40-565"><a href="de-analysis.html#cb40-565" aria-hidden="true" tabindex="-1"></a><span class="do">##   ..$ PIF                   :List of 3</span></span>
<span id="cb40-566"><a href="de-analysis.html#cb40-566" aria-hidden="true" tabindex="-1"></a><span class="do">##   ..$ DE_by_PIF_df          :List of 3</span></span>
<span id="cb40-567"><a href="de-analysis.html#cb40-567" aria-hidden="true" tabindex="-1"></a><span class="do">##  $ VMH_DESeq2_Output:List of 8</span></span>
<span id="cb40-568"><a href="de-analysis.html#cb40-568" aria-hidden="true" tabindex="-1"></a><span class="do">##   ..$ dds_wald_object       :Formal class &#39;DESeqDataSet&#39; [package &quot;DESeq2&quot;] with 8 slots</span></span>
<span id="cb40-569"><a href="de-analysis.html#cb40-569" aria-hidden="true" tabindex="-1"></a><span class="do">##   ..$ boxplot_cooks_distance:List of 9</span></span>
<span id="cb40-570"><a href="de-analysis.html#cb40-570" aria-hidden="true" tabindex="-1"></a><span class="do">##   .. ..- attr(*, &quot;class&quot;)= chr [1:2] &quot;gg&quot; &quot;ggplot&quot;</span></span>
<span id="cb40-571"><a href="de-analysis.html#cb40-571" aria-hidden="true" tabindex="-1"></a><span class="do">##   ..$ DESeq2_res_object     :List of 3</span></span>
<span id="cb40-572"><a href="de-analysis.html#cb40-572" aria-hidden="true" tabindex="-1"></a><span class="do">##   ..$ pairwise_plots        :List of 2</span></span>
<span id="cb40-573"><a href="de-analysis.html#cb40-573" aria-hidden="true" tabindex="-1"></a><span class="do">##   ..$ overall_plots         :List of 3</span></span>
<span id="cb40-574"><a href="de-analysis.html#cb40-574" aria-hidden="true" tabindex="-1"></a><span class="do">##   ..$ normalised_data       :List of 2</span></span>
<span id="cb40-575"><a href="de-analysis.html#cb40-575" aria-hidden="true" tabindex="-1"></a><span class="do">##   ..$ PIF                   :List of 3</span></span>
<span id="cb40-576"><a href="de-analysis.html#cb40-576" aria-hidden="true" tabindex="-1"></a><span class="do">##   ..$ DE_by_PIF_df          :List of 3</span></span>
<span id="cb40-577"><a href="de-analysis.html#cb40-577" aria-hidden="true" tabindex="-1"></a><span class="do">##  $ ABO_DESeq2_Output:List of 8</span></span>
<span id="cb40-578"><a href="de-analysis.html#cb40-578" aria-hidden="true" tabindex="-1"></a><span class="do">##   ..$ dds_wald_object       :Formal class &#39;DESeqDataSet&#39; [package &quot;DESeq2&quot;] with 8 slots</span></span>
<span id="cb40-579"><a href="de-analysis.html#cb40-579" aria-hidden="true" tabindex="-1"></a><span class="do">##   ..$ boxplot_cooks_distance:List of 9</span></span>
<span id="cb40-580"><a href="de-analysis.html#cb40-580" aria-hidden="true" tabindex="-1"></a><span class="do">##   .. ..- attr(*, &quot;class&quot;)= chr [1:2] &quot;gg&quot; &quot;ggplot&quot;</span></span>
<span id="cb40-581"><a href="de-analysis.html#cb40-581" aria-hidden="true" tabindex="-1"></a><span class="do">##   ..$ DESeq2_res_object     :List of 3</span></span>
<span id="cb40-582"><a href="de-analysis.html#cb40-582" aria-hidden="true" tabindex="-1"></a><span class="do">##   ..$ pairwise_plots        :List of 2</span></span>
<span id="cb40-583"><a href="de-analysis.html#cb40-583" aria-hidden="true" tabindex="-1"></a><span class="do">##   ..$ overall_plots         :List of 3</span></span>
<span id="cb40-584"><a href="de-analysis.html#cb40-584" aria-hidden="true" tabindex="-1"></a><span class="do">##   ..$ normalised_data       :List of 2</span></span>
<span id="cb40-585"><a href="de-analysis.html#cb40-585" aria-hidden="true" tabindex="-1"></a><span class="do">##   ..$ PIF                   :List of 3</span></span>
<span id="cb40-586"><a href="de-analysis.html#cb40-586" aria-hidden="true" tabindex="-1"></a><span class="do">##   ..$ DE_by_PIF_df          :List of 3</span></span>
<span id="cb40-587"><a href="de-analysis.html#cb40-587" aria-hidden="true" tabindex="-1"></a><span class="do">##  $ DUO_DESeq2_Output:List of 8</span></span>
<span id="cb40-588"><a href="de-analysis.html#cb40-588" aria-hidden="true" tabindex="-1"></a><span class="do">##   ..$ dds_wald_object       :Formal class &#39;DESeqDataSet&#39; [package &quot;DESeq2&quot;] with 8 slots</span></span>
<span id="cb40-589"><a href="de-analysis.html#cb40-589" aria-hidden="true" tabindex="-1"></a><span class="do">##   ..$ boxplot_cooks_distance:List of 9</span></span>
<span id="cb40-590"><a href="de-analysis.html#cb40-590" aria-hidden="true" tabindex="-1"></a><span class="do">##   .. ..- attr(*, &quot;class&quot;)= chr [1:2] &quot;gg&quot; &quot;ggplot&quot;</span></span>
<span id="cb40-591"><a href="de-analysis.html#cb40-591" aria-hidden="true" tabindex="-1"></a><span class="do">##   ..$ DESeq2_res_object     :List of 3</span></span>
<span id="cb40-592"><a href="de-analysis.html#cb40-592" aria-hidden="true" tabindex="-1"></a><span class="do">##   ..$ pairwise_plots        :List of 2</span></span>
<span id="cb40-593"><a href="de-analysis.html#cb40-593" aria-hidden="true" tabindex="-1"></a><span class="do">##   ..$ overall_plots         :List of 3</span></span>
<span id="cb40-594"><a href="de-analysis.html#cb40-594" aria-hidden="true" tabindex="-1"></a><span class="do">##   ..$ normalised_data       :List of 2</span></span>
<span id="cb40-595"><a href="de-analysis.html#cb40-595" aria-hidden="true" tabindex="-1"></a><span class="do">##   ..$ PIF                   :List of 3</span></span>
<span id="cb40-596"><a href="de-analysis.html#cb40-596" aria-hidden="true" tabindex="-1"></a><span class="do">##   ..$ DE_by_PIF_df          :List of 3</span></span>
<span id="cb40-597"><a href="de-analysis.html#cb40-597" aria-hidden="true" tabindex="-1"></a><span class="do">##  $ RUM_DESeq2_Output:List of 8</span></span>
<span id="cb40-598"><a href="de-analysis.html#cb40-598" aria-hidden="true" tabindex="-1"></a><span class="do">##   ..$ dds_wald_object       :Formal class &#39;DESeqDataSet&#39; [package &quot;DESeq2&quot;] with 8 slots</span></span>
<span id="cb40-599"><a href="de-analysis.html#cb40-599" aria-hidden="true" tabindex="-1"></a><span class="do">##   ..$ boxplot_cooks_distance:List of 9</span></span>
<span id="cb40-600"><a href="de-analysis.html#cb40-600" aria-hidden="true" tabindex="-1"></a><span class="do">##   .. ..- attr(*, &quot;class&quot;)= chr [1:2] &quot;gg&quot; &quot;ggplot&quot;</span></span>
<span id="cb40-601"><a href="de-analysis.html#cb40-601" aria-hidden="true" tabindex="-1"></a><span class="do">##   ..$ DESeq2_res_object     :List of 3</span></span>
<span id="cb40-602"><a href="de-analysis.html#cb40-602" aria-hidden="true" tabindex="-1"></a><span class="do">##   ..$ pairwise_plots        :List of 2</span></span>
<span id="cb40-603"><a href="de-analysis.html#cb40-603" aria-hidden="true" tabindex="-1"></a><span class="do">##   ..$ overall_plots         :List of 3</span></span>
<span id="cb40-604"><a href="de-analysis.html#cb40-604" aria-hidden="true" tabindex="-1"></a><span class="do">##   ..$ normalised_data       :List of 2</span></span>
<span id="cb40-605"><a href="de-analysis.html#cb40-605" aria-hidden="true" tabindex="-1"></a><span class="do">##   ..$ PIF                   :List of 3</span></span>
<span id="cb40-606"><a href="de-analysis.html#cb40-606" aria-hidden="true" tabindex="-1"></a><span class="do">##   ..$ DE_by_PIF_df          :List of 3</span></span>
<span id="cb40-607"><a href="de-analysis.html#cb40-607" aria-hidden="true" tabindex="-1"></a><span class="do">##  $ LIV_DESeq2_Output:List of 8</span></span>
<span id="cb40-608"><a href="de-analysis.html#cb40-608" aria-hidden="true" tabindex="-1"></a><span class="do">##   ..$ dds_wald_object       :Formal class &#39;DESeqDataSet&#39; [package &quot;DESeq2&quot;] with 8 slots</span></span>
<span id="cb40-609"><a href="de-analysis.html#cb40-609" aria-hidden="true" tabindex="-1"></a><span class="do">##   ..$ boxplot_cooks_distance:List of 9</span></span>
<span id="cb40-610"><a href="de-analysis.html#cb40-610" aria-hidden="true" tabindex="-1"></a><span class="do">##   .. ..- attr(*, &quot;class&quot;)= chr [1:2] &quot;gg&quot; &quot;ggplot&quot;</span></span>
<span id="cb40-611"><a href="de-analysis.html#cb40-611" aria-hidden="true" tabindex="-1"></a><span class="do">##   ..$ DESeq2_res_object     :List of 3</span></span>
<span id="cb40-612"><a href="de-analysis.html#cb40-612" aria-hidden="true" tabindex="-1"></a><span class="do">##   ..$ pairwise_plots        :List of 2</span></span>
<span id="cb40-613"><a href="de-analysis.html#cb40-613" aria-hidden="true" tabindex="-1"></a><span class="do">##   ..$ overall_plots         :List of 3</span></span>
<span id="cb40-614"><a href="de-analysis.html#cb40-614" aria-hidden="true" tabindex="-1"></a><span class="do">##   ..$ normalised_data       :List of 2</span></span>
<span id="cb40-615"><a href="de-analysis.html#cb40-615" aria-hidden="true" tabindex="-1"></a><span class="do">##   ..$ PIF                   :List of 3</span></span>
<span id="cb40-616"><a href="de-analysis.html#cb40-616" aria-hidden="true" tabindex="-1"></a><span class="do">##   ..$ DE_by_PIF_df          :List of 3</span></span></code></pre></div>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="prepare-data.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="access-results.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/USERNAME/REPO/edit/BRANCH/05-DE_analysis.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["_main.pdf"],
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
