<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 11 PCIT | Gene expression analysis using the DE4Rumi package</title>
  <meta name="description" content="<p>Semi-automatic analysis of RNA-seq datasets, incorporating DESeq2, PIF, RIF,
and PCIT. Designed and tested on ruminant datasets that typically include
RNA-seq data form multiple tissue regions and test between multiple pairwise
treatment comparisons.</p>" />
  <meta name="generator" content="bookdown 0.24 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 11 PCIT | Gene expression analysis using the DE4Rumi package" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="<p>Semi-automatic analysis of RNA-seq datasets, incorporating DESeq2, PIF, RIF,
and PCIT. Designed and tested on ruminant datasets that typically include
RNA-seq data form multiple tissue regions and test between multiple pairwise
treatment comparisons.</p>" />
  <meta name="github-repo" content="davos-i/DE4Rumi_Instructions" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 11 PCIT | Gene expression analysis using the DE4Rumi package" />
  
  <meta name="twitter:description" content="<p>Semi-automatic analysis of RNA-seq datasets, incorporating DESeq2, PIF, RIF,
and PCIT. Designed and tested on ruminant datasets that typically include
RNA-seq data form multiple tissue regions and test between multiple pairwise
treatment comparisons.</p>" />
  

<meta name="author" content="David Innes" />


<meta name="date" content="2022-01-06" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="rif-analysis.html"/>
<link rel="next" href="plot-genes-of-interest.html"/>
<script src="libs/header-attrs-2.11/header-attrs.js"></script>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Gene expression analysis with DE4Rumi</a></li>

<li class="divider"></li>
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Introduction</a>
<ul>
<li class="chapter" data-level="1.1" data-path="index.html"><a href="index.html#installing-de4rumi-package"><i class="fa fa-check"></i><b>1.1</b> Installing DE4Rumi Package</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="project-set-up.html"><a href="project-set-up.html"><i class="fa fa-check"></i><b>2</b> Project set up</a>
<ul>
<li class="chapter" data-level="2.1" data-path="project-set-up.html"><a href="project-set-up.html#use-r-notebooks"><i class="fa fa-check"></i><b>2.1</b> Use R Notebooks</a></li>
<li class="chapter" data-level="2.2" data-path="project-set-up.html"><a href="project-set-up.html#setup-packages"><i class="fa fa-check"></i><b>2.2</b> Setup packages</a></li>
<li class="chapter" data-level="2.3" data-path="project-set-up.html"><a href="project-set-up.html#a-note-on-file-paths"><i class="fa fa-check"></i><b>2.3</b> A note on file paths</a></li>
<li class="chapter" data-level="2.4" data-path="project-set-up.html"><a href="project-set-up.html#figures-displayed-below-code-chunks"><i class="fa fa-check"></i><b>2.4</b> Figures displayed below code chunks</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="importing-data.html"><a href="importing-data.html"><i class="fa fa-check"></i><b>3</b> Importing Data</a>
<ul>
<li class="chapter" data-level="3.1" data-path="importing-data.html"><a href="importing-data.html#note-on-importing-files"><i class="fa fa-check"></i><b>3.1</b> Note on importing files</a></li>
<li class="chapter" data-level="3.2" data-path="importing-data.html"><a href="importing-data.html#import-count-data"><i class="fa fa-check"></i><b>3.2</b> Import count data</a>
<ul>
<li class="chapter" data-level="3.2.1" data-path="importing-data.html"><a href="importing-data.html#check-structure-of-data"><i class="fa fa-check"></i><b>3.2.1</b> Check structure of data</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="importing-data.html"><a href="importing-data.html#import-column-metadata"><i class="fa fa-check"></i><b>3.3</b> Import column metadata</a></li>
<li class="chapter" data-level="3.4" data-path="importing-data.html"><a href="importing-data.html#rename-columns-to-match-required-format"><i class="fa fa-check"></i><b>3.4</b> Rename columns to match required format</a></li>
<li class="chapter" data-level="3.5" data-path="importing-data.html"><a href="importing-data.html#check-data"><i class="fa fa-check"></i><b>3.5</b> Check data</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="prepare-data.html"><a href="prepare-data.html"><i class="fa fa-check"></i><b>4</b> Prepare Data</a>
<ul>
<li class="chapter" data-level="4.1" data-path="prepare-data.html"><a href="prepare-data.html#get-human-equivalant-gene-names"><i class="fa fa-check"></i><b>4.1</b> Get human equivalant gene names</a></li>
<li class="chapter" data-level="4.2" data-path="prepare-data.html"><a href="prepare-data.html#prepare-summarized-experiment-object"><i class="fa fa-check"></i><b>4.2</b> Prepare summarized experiment object</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="de-analysis.html"><a href="de-analysis.html"><i class="fa fa-check"></i><b>5</b> DE Analysis</a>
<ul>
<li class="chapter" data-level="5.1" data-path="de-analysis.html"><a href="de-analysis.html#generate-de-results"><i class="fa fa-check"></i><b>5.1</b> Generate DE results</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="access-results.html"><a href="access-results.html"><i class="fa fa-check"></i><b>6</b> 6. Access results</a>
<ul>
<li class="chapter" data-level="6.1" data-path="access-results.html"><a href="access-results.html#boxplot-of-cooks-distances"><i class="fa fa-check"></i><b>6.1</b> Boxplot of cooks distances</a></li>
<li class="chapter" data-level="6.2" data-path="access-results.html"><a href="access-results.html#pca"><i class="fa fa-check"></i><b>6.2</b> PCA</a></li>
<li class="chapter" data-level="6.3" data-path="access-results.html"><a href="access-results.html#heatmap"><i class="fa fa-check"></i><b>6.3</b> Heatmap</a></li>
<li class="chapter" data-level="6.4" data-path="access-results.html"><a href="access-results.html#sample-distances"><i class="fa fa-check"></i><b>6.4</b> Sample distances</a></li>
<li class="chapter" data-level="6.5" data-path="access-results.html"><a href="access-results.html#remove-outliers-if-needed-and-re-run-steps-de-analysis"><i class="fa fa-check"></i><b>6.5</b> Remove outliers if needed and re-run steps DE Analysis</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="functional-enrichment-analysis.html"><a href="functional-enrichment-analysis.html"><i class="fa fa-check"></i><b>7</b> Functional enrichment analysis</a></li>
<li class="chapter" data-level="8" data-path="ma-plots.html"><a href="ma-plots.html"><i class="fa fa-check"></i><b>8</b> MA Plots</a></li>
<li class="chapter" data-level="9" data-path="whole-dataset-normalisation.html"><a href="whole-dataset-normalisation.html"><i class="fa fa-check"></i><b>9</b> Whole dataset normalisation</a>
<ul>
<li class="chapter" data-level="9.1" data-path="whole-dataset-normalisation.html"><a href="whole-dataset-normalisation.html#view-cooks-distance-boxplot-for-whole-dataset"><i class="fa fa-check"></i><b>9.1</b> View Cooks distance boxplot for whole dataset</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="rif-analysis.html"><a href="rif-analysis.html"><i class="fa fa-check"></i><b>10</b> RIF Analysis</a></li>
<li class="chapter" data-level="11" data-path="pcit.html"><a href="pcit.html"><i class="fa fa-check"></i><b>11</b> PCIT</a>
<ul>
<li class="chapter" data-level="11.1" data-path="pcit.html"><a href="pcit.html#prepare-data-for-pcit-analysis"><i class="fa fa-check"></i><b>11.1</b> Prepare data for PCIT analysis</a></li>
<li class="chapter" data-level="11.2" data-path="pcit.html"><a href="pcit.html#run-pcit-analysis"><i class="fa fa-check"></i><b>11.2</b> Run PCIT Analysis</a></li>
<li class="chapter" data-level="11.3" data-path="pcit.html"><a href="pcit.html#filter-pcit-output-for-cytoscape"><i class="fa fa-check"></i><b>11.3</b> Filter PCIT output for Cytoscape</a></li>
<li class="chapter" data-level="11.4" data-path="pcit.html"><a href="pcit.html#viewing-in-cytoscape"><i class="fa fa-check"></i><b>11.4</b> Viewing in Cytoscape</a>
<ul>
<li class="chapter" data-level="11.4.1" data-path="pcit.html"><a href="pcit.html#import-pcit-output-to-cytoscape"><i class="fa fa-check"></i><b>11.4.1</b> Import PCIT output to cytoscape</a></li>
<li class="chapter" data-level="11.4.2" data-path="pcit.html"><a href="pcit.html#import-metadata-for-genes"><i class="fa fa-check"></i><b>11.4.2</b> Import metadata for genes</a></li>
<li class="chapter" data-level="11.4.3" data-path="pcit.html"><a href="pcit.html#further-details"><i class="fa fa-check"></i><b>11.4.3</b> Further Details</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="12" data-path="plot-genes-of-interest.html"><a href="plot-genes-of-interest.html"><i class="fa fa-check"></i><b>12</b> Plot genes of interest</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Gene expression analysis using the DE4Rumi package</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="pcit" class="section level1" number="11">
<h1><span class="header-section-number">Chapter 11</span> PCIT</h1>
<p>The Partial Correlation Information Theory (PCIT) co-expression analysis is a robust
technique for determing biological meaningful correlations between genes across the variability within a dataset.</p>
<div id="prepare-data-for-pcit-analysis" class="section level2" number="11.1">
<h2><span class="header-section-number">11.1</span> Prepare data for PCIT analysis</h2>
<p>It is prohibitive to run PCIT on all possible genes, are largely unmeaningful. Instead, the first step in this analysis filters each set of data to include a list of genes that are most likely to be meaningful in a co-expression analysis. Three possible data types can be included: DE, PIF and RIF. For each, the function will use user-defined thresholds to make selections for each pairwise comparison. In this example a small number has been selected to keep the computational load to a minimum. In reality, larger <code>DEn</code>, <code>PIFn</code> and <code>RIFn</code> values might be used.</p>
<p>This function will output a venn plot which is helpful to see how many genes from each catagory were selected. It will also output metadata which is useful for visualisation in cytoscape.</p>
<div class="sourceCode" id="cb52"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb52-1"><a href="pcit.html#cb52-1" aria-hidden="true" tabindex="-1"></a>vst_norm_data <span class="ot">&lt;-</span> <span class="fu">GET_normalised_data</span>(full_norm, <span class="at">vst_or_log2 =</span> <span class="st">&quot;VST&quot;</span>)[[<span class="dv">1</span>]]</span>
<span id="cb52-2"><a href="pcit.html#cb52-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb52-3"><a href="pcit.html#cb52-3" aria-hidden="true" tabindex="-1"></a>selected_for_PCIT <span class="ot">&lt;-</span> </span>
<span id="cb52-4"><a href="pcit.html#cb52-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">PCIT_prepare_data</span>(<span class="at">data_type_selection =</span> <span class="fu">c</span>(<span class="st">&quot;DE&quot;</span>, <span class="st">&quot;PIF&quot;</span>, <span class="st">&quot;RIF&quot;</span>), </span>
<span id="cb52-5"><a href="pcit.html#cb52-5" aria-hidden="true" tabindex="-1"></a>                    <span class="at">norm_exp_data =</span> vst_norm_data,</span>
<span id="cb52-6"><a href="pcit.html#cb52-6" aria-hidden="true" tabindex="-1"></a>                    <span class="at">gene_annotations =</span> gene_annot,</span>
<span id="cb52-7"><a href="pcit.html#cb52-7" aria-hidden="true" tabindex="-1"></a>                    <span class="at">DE_full_out =</span> DE_out,</span>
<span id="cb52-8"><a href="pcit.html#cb52-8" aria-hidden="true" tabindex="-1"></a>                    <span class="at">RIF_output_data =</span> RIF_output,</span>
<span id="cb52-9"><a href="pcit.html#cb52-9" aria-hidden="true" tabindex="-1"></a>                    <span class="at">search_top_level =</span> <span class="cn">NULL</span>,</span>
<span id="cb52-10"><a href="pcit.html#cb52-10" aria-hidden="true" tabindex="-1"></a>                    <span class="at">filter_norm_data_columns =</span> <span class="cn">TRUE</span>,</span>
<span id="cb52-11"><a href="pcit.html#cb52-11" aria-hidden="true" tabindex="-1"></a>                    <span class="at">DEn =</span> <span class="dv">20</span> ,</span>
<span id="cb52-12"><a href="pcit.html#cb52-12" aria-hidden="true" tabindex="-1"></a>                    <span class="at">PIF_sig =</span> <span class="fl">2.58</span>,</span>
<span id="cb52-13"><a href="pcit.html#cb52-13" aria-hidden="true" tabindex="-1"></a>                    <span class="at">PIFn =</span> <span class="dv">10</span>,</span>
<span id="cb52-14"><a href="pcit.html#cb52-14" aria-hidden="true" tabindex="-1"></a>                    <span class="at">RIF_sig =</span> <span class="fl">2.58</span>,</span>
<span id="cb52-15"><a href="pcit.html#cb52-15" aria-hidden="true" tabindex="-1"></a>                    <span class="at">RIFn =</span> <span class="dv">10</span>, </span>
<span id="cb52-16"><a href="pcit.html#cb52-16" aria-hidden="true" tabindex="-1"></a>                    <span class="at">export_tables =</span> <span class="cn">TRUE</span>,</span>
<span id="cb52-17"><a href="pcit.html#cb52-17" aria-hidden="true" tabindex="-1"></a>                    <span class="at">export_venn_plot =</span> <span class="cn">TRUE</span>,</span>
<span id="cb52-18"><a href="pcit.html#cb52-18" aria-hidden="true" tabindex="-1"></a>                    <span class="at">export_dir =</span> <span class="st">&quot;./outputs/PCIT/&quot;</span>)</span></code></pre></div>
<pre><code>## Number of selected genes within dataset:  771</code></pre>
<pre><code>## Number of samples within dataset:  105</code></pre>
<pre><code>## Directory created: ./outputs/PCIT/</code></pre>
</div>
<div id="run-pcit-analysis" class="section level2" number="11.2">
<h2><span class="header-section-number">11.2</span> Run PCIT Analysis</h2>
<p>This step may take a while, and it may look like R is not doing anything as it is implemented in C in the background. If you have access to the windows exectuable fortran program it would likely be quicker to use that instead.</p>
<p>This function will output a list of tables with different calculated values.</p>
<div class="sourceCode" id="cb56"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb56-1"><a href="pcit.html#cb56-1" aria-hidden="true" tabindex="-1"></a>PCIT_norm_counts <span class="ot">&lt;-</span> selected_for_PCIT<span class="sc">$</span>PCIT_normalised_counts</span>
<span id="cb56-2"><a href="pcit.html#cb56-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb56-3"><a href="pcit.html#cb56-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb56-4"><a href="pcit.html#cb56-4" aria-hidden="true" tabindex="-1"></a>PCIT_out <span class="ot">&lt;-</span> </span>
<span id="cb56-5"><a href="pcit.html#cb56-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">PCIT_calculate</span>(<span class="at">norm_counts_for_PCIT =</span> PCIT_norm_counts,</span>
<span id="cb56-6"><a href="pcit.html#cb56-6" aria-hidden="true" tabindex="-1"></a>                 <span class="at">key_colname =</span> <span class="st">&quot;gene_name&quot;</span>)</span></code></pre></div>
<pre><code>## Removing rows with duplicated rownames: U6, U6, LYZ1, U6, ITLN2</code></pre>
<div class="sourceCode" id="cb58"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb58-1"><a href="pcit.html#cb58-1" aria-hidden="true" tabindex="-1"></a><span class="co">#Check output table</span></span>
<span id="cb58-2"><a href="pcit.html#cb58-2" aria-hidden="true" tabindex="-1"></a>PCIT_out<span class="sc">$</span>tab <span class="sc">%&gt;%</span> <span class="fu">head</span>()</span></code></pre></div>
<pre><code>##       gene1              gene2    corr1 corr2
## 1 5_8S_rRNA              ITLN2  0.13753     0
## 2 5_8S_rRNA ENSBTAG00000048770 -0.17807     0
## 3 5_8S_rRNA ENSBTAG00000048794  0.12162     0
## 4 5_8S_rRNA ENSBTAG00000048804 -0.13235     0
## 5 5_8S_rRNA ENSBTAG00000048850 -0.10011     0
## 6 5_8S_rRNA ENSBTAG00000048871  0.24243     0</code></pre>
</div>
<div id="filter-pcit-output-for-cytoscape" class="section level2" number="11.3">
<h2><span class="header-section-number">11.3</span> Filter PCIT output for Cytoscape</h2>
<p>The output from the PCIT function is not filtered and is also not in a format compatible with Cytoscape software. The <code>PCIT_filter_output()</code> function handles this. It is not always known what the best threshold will be for visualisation, and this step is likely to be done multiple times in practice.</p>
<p>This function will output the required files for visualisation in Cytoscape.</p>
<div class="sourceCode" id="cb60"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb60-1"><a href="pcit.html#cb60-1" aria-hidden="true" tabindex="-1"></a><span class="fu">PCIT_filter_output</span>(<span class="at">PCIT_out_data =</span> PCIT_out,</span>
<span id="cb60-2"><a href="pcit.html#cb60-2" aria-hidden="true" tabindex="-1"></a>                   <span class="at">cor_threshold =</span> <span class="fl">0.8</span>,</span>
<span id="cb60-3"><a href="pcit.html#cb60-3" aria-hidden="true" tabindex="-1"></a>                   <span class="at">export_dir =</span> <span class="st">&quot;./outputs/PCIT/&quot;</span>)</span>
<span id="cb60-4"><a href="pcit.html#cb60-4" aria-hidden="true" tabindex="-1"></a><span class="do">## Registered S3 method overwritten by &#39;GGally&#39;:</span></span>
<span id="cb60-5"><a href="pcit.html#cb60-5" aria-hidden="true" tabindex="-1"></a><span class="do">##   method from   </span></span>
<span id="cb60-6"><a href="pcit.html#cb60-6" aria-hidden="true" tabindex="-1"></a><span class="do">##   +.gg   ggplot2</span></span>
<span id="cb60-7"><a href="pcit.html#cb60-7" aria-hidden="true" tabindex="-1"></a><span class="do">## </span></span>
<span id="cb60-8"><a href="pcit.html#cb60-8" aria-hidden="true" tabindex="-1"></a><span class="do">## Registered S3 method overwritten by &#39;ggnetwork&#39;:</span></span>
<span id="cb60-9"><a href="pcit.html#cb60-9" aria-hidden="true" tabindex="-1"></a><span class="do">##   method         from  </span></span>
<span id="cb60-10"><a href="pcit.html#cb60-10" aria-hidden="true" tabindex="-1"></a><span class="do">##   fortify.igraph ggtree</span></span>
<span id="cb60-11"><a href="pcit.html#cb60-11" aria-hidden="true" tabindex="-1"></a><span class="do">## number of nodes:</span></span>
<span id="cb60-12"><a href="pcit.html#cb60-12" aria-hidden="true" tabindex="-1"></a><span class="do">## 498</span></span>
<span id="cb60-13"><a href="pcit.html#cb60-13" aria-hidden="true" tabindex="-1"></a><span class="do">## ./outputs/PCIT/ Directory exists</span></span>
<span id="cb60-14"><a href="pcit.html#cb60-14" aria-hidden="true" tabindex="-1"></a><span class="do">## Exported data to : ./outputs/PCIT/PCIT_FILTERED_0.8_20220106_2259.txt</span></span>
<span id="cb60-15"><a href="pcit.html#cb60-15" aria-hidden="true" tabindex="-1"></a><span class="do">## Creating summary density plots...</span></span></code></pre></div>
<p><img src="_main_files/figure-html/unnamed-chunk-31-1.png" width="672" /></p>
<div class="sourceCode" id="cb61"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb61-1"><a href="pcit.html#cb61-1" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb61-2"><a href="pcit.html#cb61-2" aria-hidden="true" tabindex="-1"></a><span class="fu">PCIT_filter_output</span>(<span class="at">PCIT_out_data =</span> PCIT_out,</span>
<span id="cb61-3"><a href="pcit.html#cb61-3" aria-hidden="true" tabindex="-1"></a>                   <span class="at">cor_threshold =</span> <span class="fl">0.9</span>,</span>
<span id="cb61-4"><a href="pcit.html#cb61-4" aria-hidden="true" tabindex="-1"></a>                   <span class="at">export_dir =</span> <span class="st">&quot;./outputs/PCIT/&quot;</span>)</span>
<span id="cb61-5"><a href="pcit.html#cb61-5" aria-hidden="true" tabindex="-1"></a><span class="do">## number of nodes:</span></span>
<span id="cb61-6"><a href="pcit.html#cb61-6" aria-hidden="true" tabindex="-1"></a><span class="do">## 264</span></span>
<span id="cb61-7"><a href="pcit.html#cb61-7" aria-hidden="true" tabindex="-1"></a><span class="do">## ./outputs/PCIT/ Directory exists</span></span>
<span id="cb61-8"><a href="pcit.html#cb61-8" aria-hidden="true" tabindex="-1"></a><span class="do">## Exported data to : ./outputs/PCIT/PCIT_FILTERED_0.9_20220106_2259.txt</span></span>
<span id="cb61-9"><a href="pcit.html#cb61-9" aria-hidden="true" tabindex="-1"></a><span class="do">## Creating summary density plots...</span></span></code></pre></div>
<p><img src="_main_files/figure-html/unnamed-chunk-31-2.png" width="672" /></p>
</div>
<div id="viewing-in-cytoscape" class="section level2" number="11.4">
<h2><span class="header-section-number">11.4</span> Viewing in Cytoscape</h2>
<p>Firstly, go to <a href="https://cytoscape.org/download.html" class="uri">https://cytoscape.org/download.html</a> and download Cytoscape.</p>
<p>There is also plenty of help files on their website, but a brief overview is included here.</p>
<p>The exported file from the <code>PCIT_filter_output</code> function will look something like: <code>PCIT_FILTERED_0.8_20220106_0914.txt</code>.
This file is formatted appropriately for Cytoscape to import it easily.</p>
<div id="import-pcit-output-to-cytoscape" class="section level3" number="11.4.1">
<h3><span class="header-section-number">11.4.1</span> Import PCIT output to cytoscape</h3>
<p>To import the file:</p>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:pcit1"></span>
<img src="Images/Image01.png" alt="Select File -&gt; Import -&gt; Network from File" width="100%" />
<p class="caption">
Figure 11.1: Select File -&gt; Import -&gt; Network from File
</p>
</div>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:pcit2"></span>
<img src="Images/Image02.png" alt="Select the required 'PCIT_filtered_' file and Open it" width="100%" />
<p class="caption">
Figure 11.2: Select the required ‘PCIT_filtered_’ file and Open it
</p>
</div>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:pcit3"></span>
<img src="Images/Image03.png" alt="Make sure the first column is a green circle and the second column is the orange circular symbol. Click OK." width="100%" />
<p class="caption">
Figure 11.3: Make sure the first column is a green circle and the second column is the orange circular symbol. Click OK.
</p>
</div>
<p>Once this is done you should see a display of the network being built.</p>
</div>
<div id="import-metadata-for-genes" class="section level3" number="11.4.2">
<h3><span class="header-section-number">11.4.2</span> Import metadata for genes</h3>
<p>It is also helpful to include more information about the genes in Cytoscape to help with interpretation. One way is to do the following:</p>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:pcit4"></span>
<img src="Images/Image04.png" alt="Select File -&gt; Import -&gt; Table from File" width="100%" />
<p class="caption">
Figure 11.4: Select File -&gt; Import -&gt; Table from File
</p>
</div>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:pcit5"></span>
<img src="Images/Image05.png" alt="Select the 'genes_summary' file" width="100%" />
<p class="caption">
Figure 11.5: Select the ‘genes_summary’ file
</p>
</div>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:pcit6"></span>
<img src="Images/Image06.png" alt="Change the key to match the'key_colname = ' value from PCIT function, in this case it is the gene_name column. Click on column heading to change." width="100%" />
<p class="caption">
Figure 11.6: Change the key to match the’key_colname = ’ value from PCIT function, in this case it is the gene_name column. Click on column heading to change.
</p>
</div>
</div>
<div id="further-details" class="section level3" number="11.4.3">
<h3><span class="header-section-number">11.4.3</span> Further Details</h3>
<p>Further help is available in the manual: <a href="http://manual.cytoscape.org/en/stable/" class="uri">http://manual.cytoscape.org/en/stable/</a></p>

</div>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="rif-analysis.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="plot-genes-of-interest.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/USERNAME/REPO/edit/BRANCH/10-PCIT.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["_main.pdf"],
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
