<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 9 Whole dataset normalisation | Gene expression analysis using the DE4Rumi package</title>
  <meta name="description" content="<p>Semi-automatic analysis of RNA-seq datasets, incorporating DESeq2, PIF, RIF,
and PCIT. Designed and tested on ruminant datasets that typically include
RNA-seq data form multiple tissue regions and test between multiple pairwise
treatment comparisons.</p>" />
  <meta name="generator" content="bookdown 0.24 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 9 Whole dataset normalisation | Gene expression analysis using the DE4Rumi package" />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="<p>Semi-automatic analysis of RNA-seq datasets, incorporating DESeq2, PIF, RIF,
and PCIT. Designed and tested on ruminant datasets that typically include
RNA-seq data form multiple tissue regions and test between multiple pairwise
treatment comparisons.</p>" />
  <meta name="github-repo" content="davos-i/DE4Rumi_Instructions" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 9 Whole dataset normalisation | Gene expression analysis using the DE4Rumi package" />
  
  <meta name="twitter:description" content="<p>Semi-automatic analysis of RNA-seq datasets, incorporating DESeq2, PIF, RIF,
and PCIT. Designed and tested on ruminant datasets that typically include
RNA-seq data form multiple tissue regions and test between multiple pairwise
treatment comparisons.</p>" />
  

<meta name="author" content="David Innes" />


<meta name="date" content="2022-01-06" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="ma-plots.html"/>
<link rel="next" href="rif-analysis.html"/>
<script src="libs/header-attrs-2.11/header-attrs.js"></script>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Gene expression analysis with DE4Rumi</a></li>

<li class="divider"></li>
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Introduction</a>
<ul>
<li class="chapter" data-level="1.1" data-path="index.html"><a href="index.html#installing-de4rumi-package"><i class="fa fa-check"></i><b>1.1</b> Installing DE4Rumi Package</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="project-set-up.html"><a href="project-set-up.html"><i class="fa fa-check"></i><b>2</b> Project set up</a>
<ul>
<li class="chapter" data-level="2.1" data-path="project-set-up.html"><a href="project-set-up.html#use-r-notebooks"><i class="fa fa-check"></i><b>2.1</b> Use R Notebooks</a></li>
<li class="chapter" data-level="2.2" data-path="project-set-up.html"><a href="project-set-up.html#setup-packages"><i class="fa fa-check"></i><b>2.2</b> Setup packages</a></li>
<li class="chapter" data-level="2.3" data-path="project-set-up.html"><a href="project-set-up.html#a-note-on-file-paths"><i class="fa fa-check"></i><b>2.3</b> A note on file paths</a></li>
<li class="chapter" data-level="2.4" data-path="project-set-up.html"><a href="project-set-up.html#figures-displayed-below-code-chunks"><i class="fa fa-check"></i><b>2.4</b> Figures displayed below code chunks</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="importing-data.html"><a href="importing-data.html"><i class="fa fa-check"></i><b>3</b> Importing Data</a>
<ul>
<li class="chapter" data-level="3.1" data-path="importing-data.html"><a href="importing-data.html#note-on-importing-files"><i class="fa fa-check"></i><b>3.1</b> Note on importing files</a></li>
<li class="chapter" data-level="3.2" data-path="importing-data.html"><a href="importing-data.html#import-count-data"><i class="fa fa-check"></i><b>3.2</b> Import count data</a>
<ul>
<li class="chapter" data-level="3.2.1" data-path="importing-data.html"><a href="importing-data.html#check-structure-of-data"><i class="fa fa-check"></i><b>3.2.1</b> Check structure of data</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="importing-data.html"><a href="importing-data.html#import-column-metadata"><i class="fa fa-check"></i><b>3.3</b> Import column metadata</a></li>
<li class="chapter" data-level="3.4" data-path="importing-data.html"><a href="importing-data.html#rename-columns-to-match-required-format"><i class="fa fa-check"></i><b>3.4</b> Rename columns to match required format</a></li>
<li class="chapter" data-level="3.5" data-path="importing-data.html"><a href="importing-data.html#check-data"><i class="fa fa-check"></i><b>3.5</b> Check data</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="prepare-data.html"><a href="prepare-data.html"><i class="fa fa-check"></i><b>4</b> Prepare Data</a>
<ul>
<li class="chapter" data-level="4.1" data-path="prepare-data.html"><a href="prepare-data.html#get-human-equivalant-gene-names"><i class="fa fa-check"></i><b>4.1</b> Get human equivalant gene names</a></li>
<li class="chapter" data-level="4.2" data-path="prepare-data.html"><a href="prepare-data.html#prepare-summarized-experiment-object"><i class="fa fa-check"></i><b>4.2</b> Prepare summarized experiment object</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="de-analysis.html"><a href="de-analysis.html"><i class="fa fa-check"></i><b>5</b> DE Analysis</a>
<ul>
<li class="chapter" data-level="5.1" data-path="de-analysis.html"><a href="de-analysis.html#generate-de-results"><i class="fa fa-check"></i><b>5.1</b> Generate DE results</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="access-results.html"><a href="access-results.html"><i class="fa fa-check"></i><b>6</b> 6. Access results</a>
<ul>
<li class="chapter" data-level="6.1" data-path="access-results.html"><a href="access-results.html#boxplot-of-cooks-distances"><i class="fa fa-check"></i><b>6.1</b> Boxplot of cooks distances</a></li>
<li class="chapter" data-level="6.2" data-path="access-results.html"><a href="access-results.html#pca"><i class="fa fa-check"></i><b>6.2</b> PCA</a></li>
<li class="chapter" data-level="6.3" data-path="access-results.html"><a href="access-results.html#heatmap"><i class="fa fa-check"></i><b>6.3</b> Heatmap</a></li>
<li class="chapter" data-level="6.4" data-path="access-results.html"><a href="access-results.html#sample-distances"><i class="fa fa-check"></i><b>6.4</b> Sample distances</a></li>
<li class="chapter" data-level="6.5" data-path="access-results.html"><a href="access-results.html#remove-outliers-if-needed-and-re-run-steps-de-analysis"><i class="fa fa-check"></i><b>6.5</b> Remove outliers if needed and re-run steps DE Analysis</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="functional-enrichment-analysis.html"><a href="functional-enrichment-analysis.html"><i class="fa fa-check"></i><b>7</b> Functional enrichment analysis</a></li>
<li class="chapter" data-level="8" data-path="ma-plots.html"><a href="ma-plots.html"><i class="fa fa-check"></i><b>8</b> MA Plots</a></li>
<li class="chapter" data-level="9" data-path="whole-dataset-normalisation.html"><a href="whole-dataset-normalisation.html"><i class="fa fa-check"></i><b>9</b> Whole dataset normalisation</a>
<ul>
<li class="chapter" data-level="9.1" data-path="whole-dataset-normalisation.html"><a href="whole-dataset-normalisation.html#view-cooks-distance-boxplot-for-whole-dataset"><i class="fa fa-check"></i><b>9.1</b> View Cooks distance boxplot for whole dataset</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="rif-analysis.html"><a href="rif-analysis.html"><i class="fa fa-check"></i><b>10</b> RIF Analysis</a></li>
<li class="chapter" data-level="11" data-path="pcit.html"><a href="pcit.html"><i class="fa fa-check"></i><b>11</b> PCIT</a>
<ul>
<li class="chapter" data-level="11.1" data-path="pcit.html"><a href="pcit.html#prepare-data-for-pcit-analysis"><i class="fa fa-check"></i><b>11.1</b> Prepare data for PCIT analysis</a></li>
<li class="chapter" data-level="11.2" data-path="pcit.html"><a href="pcit.html#run-pcit-analysis"><i class="fa fa-check"></i><b>11.2</b> Run PCIT Analysis</a></li>
<li class="chapter" data-level="11.3" data-path="pcit.html"><a href="pcit.html#filter-pcit-output-for-cytoscape"><i class="fa fa-check"></i><b>11.3</b> Filter PCIT output for Cytoscape</a></li>
<li class="chapter" data-level="11.4" data-path="pcit.html"><a href="pcit.html#viewing-in-cytoscape"><i class="fa fa-check"></i><b>11.4</b> Viewing in Cytoscape</a>
<ul>
<li class="chapter" data-level="11.4.1" data-path="pcit.html"><a href="pcit.html#import-pcit-output-to-cytoscape"><i class="fa fa-check"></i><b>11.4.1</b> Import PCIT output to cytoscape</a></li>
<li class="chapter" data-level="11.4.2" data-path="pcit.html"><a href="pcit.html#import-metadata-for-genes"><i class="fa fa-check"></i><b>11.4.2</b> Import metadata for genes</a></li>
<li class="chapter" data-level="11.4.3" data-path="pcit.html"><a href="pcit.html#further-details"><i class="fa fa-check"></i><b>11.4.3</b> Further Details</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="12" data-path="plot-genes-of-interest.html"><a href="plot-genes-of-interest.html"><i class="fa fa-check"></i><b>12</b> Plot genes of interest</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Gene expression analysis using the DE4Rumi package</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="whole-dataset-normalisation" class="section level1" number="9">
<h1><span class="header-section-number">Chapter 9</span> Whole dataset normalisation</h1>
<p>After the DE analysis is complete, it is important to normalise the data across
the entire data set for downstream co-expression analysis (PCIT) and other analyses.</p>
<p>This uses the same <code>auto_generate_DE_results()</code> function but with
<code>whole_dataset_normalisation = TRUE</code>. The example below shows the minimum
information the function needs to run.</p>
<div class="sourceCode" id="cb47"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb47-1"><a href="whole-dataset-normalisation.html#cb47-1" aria-hidden="true" tabindex="-1"></a>full_norm <span class="ot">&lt;-</span> </span>
<span id="cb47-2"><a href="whole-dataset-normalisation.html#cb47-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">auto_generate_DE_results</span>(<span class="at">se_data =</span> seq_data, </span>
<span id="cb47-3"><a href="whole-dataset-normalisation.html#cb47-3" aria-hidden="true" tabindex="-1"></a>                           <span class="at">top_level_colname =</span> Tissue_Region, </span>
<span id="cb47-4"><a href="whole-dataset-normalisation.html#cb47-4" aria-hidden="true" tabindex="-1"></a>                           <span class="at">sample_colname =</span> sample_names,</span>
<span id="cb47-5"><a href="whole-dataset-normalisation.html#cb47-5" aria-hidden="true" tabindex="-1"></a>                           <span class="at">samples_to_remove =</span> <span class="cn">NA</span>,</span>
<span id="cb47-6"><a href="whole-dataset-normalisation.html#cb47-6" aria-hidden="true" tabindex="-1"></a>                           <span class="at">DESeq2_formula_design =</span> <span class="sc">~</span>Treatment,</span>
<span id="cb47-7"><a href="whole-dataset-normalisation.html#cb47-7" aria-hidden="true" tabindex="-1"></a>                           <span class="at">gene_annotations =</span> gene_annot,</span>
<span id="cb47-8"><a href="whole-dataset-normalisation.html#cb47-8" aria-hidden="true" tabindex="-1"></a>                           <span class="at">export_tables =</span> <span class="cn">TRUE</span>,</span>
<span id="cb47-9"><a href="whole-dataset-normalisation.html#cb47-9" aria-hidden="true" tabindex="-1"></a>                           <span class="at">export_dir =</span> <span class="st">&quot;./outputs/&quot;</span>,</span>
<span id="cb47-10"><a href="whole-dataset-normalisation.html#cb47-10" aria-hidden="true" tabindex="-1"></a>                           <span class="at">whole_data_normalisation =</span> <span class="cn">TRUE</span>)</span>
<span id="cb47-11"><a href="whole-dataset-normalisation.html#cb47-11" aria-hidden="true" tabindex="-1"></a><span class="do">## Whole data normalisation selected. No pairwise results will be generated.</span></span>
<span id="cb47-12"><a href="whole-dataset-normalisation.html#cb47-12" aria-hidden="true" tabindex="-1"></a><span class="do">## renaming the first element in assays to &#39;counts&#39;</span></span>
<span id="cb47-13"><a href="whole-dataset-normalisation.html#cb47-13" aria-hidden="true" tabindex="-1"></a><span class="do">## Warning in DESeq2::DESeqDataSet(se_data0, design = DESeq2_formula_design): some</span></span>
<span id="cb47-14"><a href="whole-dataset-normalisation.html#cb47-14" aria-hidden="true" tabindex="-1"></a><span class="do">## variables in design formula are characters, converting to factors</span></span>
<span id="cb47-15"><a href="whole-dataset-normalisation.html#cb47-15" aria-hidden="true" tabindex="-1"></a><span class="do">## Beginning DESeq analysis...</span></span>
<span id="cb47-16"><a href="whole-dataset-normalisation.html#cb47-16" aria-hidden="true" tabindex="-1"></a><span class="do">## estimating size factors</span></span>
<span id="cb47-17"><a href="whole-dataset-normalisation.html#cb47-17" aria-hidden="true" tabindex="-1"></a><span class="do">## estimating dispersions</span></span>
<span id="cb47-18"><a href="whole-dataset-normalisation.html#cb47-18" aria-hidden="true" tabindex="-1"></a><span class="do">## gene-wise dispersion estimates</span></span>
<span id="cb47-19"><a href="whole-dataset-normalisation.html#cb47-19" aria-hidden="true" tabindex="-1"></a><span class="do">## mean-dispersion relationship</span></span>
<span id="cb47-20"><a href="whole-dataset-normalisation.html#cb47-20" aria-hidden="true" tabindex="-1"></a><span class="do">## final dispersion estimates</span></span>
<span id="cb47-21"><a href="whole-dataset-normalisation.html#cb47-21" aria-hidden="true" tabindex="-1"></a><span class="do">## fitting model and testing</span></span>
<span id="cb47-22"><a href="whole-dataset-normalisation.html#cb47-22" aria-hidden="true" tabindex="-1"></a><span class="do">## -- replacing outliers and refitting for 398 genes</span></span>
<span id="cb47-23"><a href="whole-dataset-normalisation.html#cb47-23" aria-hidden="true" tabindex="-1"></a><span class="do">## -- DESeq argument &#39;minReplicatesForReplace&#39; = 6 </span></span>
<span id="cb47-24"><a href="whole-dataset-normalisation.html#cb47-24" aria-hidden="true" tabindex="-1"></a><span class="do">## -- original counts are preserved in counts(dds)</span></span>
<span id="cb47-25"><a href="whole-dataset-normalisation.html#cb47-25" aria-hidden="true" tabindex="-1"></a><span class="do">## estimating dispersions</span></span>
<span id="cb47-26"><a href="whole-dataset-normalisation.html#cb47-26" aria-hidden="true" tabindex="-1"></a><span class="do">## fitting model and testing</span></span>
<span id="cb47-27"><a href="whole-dataset-normalisation.html#cb47-27" aria-hidden="true" tabindex="-1"></a><span class="do">## Completed DESeq analysis.</span></span>
<span id="cb47-28"><a href="whole-dataset-normalisation.html#cb47-28" aria-hidden="true" tabindex="-1"></a><span class="do">## Plotting cooks distance...</span></span>
<span id="cb47-29"><a href="whole-dataset-normalisation.html#cb47-29" aria-hidden="true" tabindex="-1"></a><span class="do">## Cooks distance plot complete.</span></span>
<span id="cb47-30"><a href="whole-dataset-normalisation.html#cb47-30" aria-hidden="true" tabindex="-1"></a><span class="do">## ./outputs/ Directory exists</span></span>
<span id="cb47-31"><a href="whole-dataset-normalisation.html#cb47-31" aria-hidden="true" tabindex="-1"></a><span class="do">## Normalised tables exported to the sub-directory: ./outputs/</span></span></code></pre></div>
<p><img src="_main_files/figure-html/unnamed-chunk-24-1.png" width="960" /><img src="_main_files/figure-html/unnamed-chunk-24-2.png" width="960" /></p>
<pre><code>## Preparing data for output...
## List output succesfully generated.
## 
## 
##  ******************* END *******************</code></pre>
<p><img src="_main_files/figure-html/unnamed-chunk-24-3.png" width="960" /></p>
<div id="view-cooks-distance-boxplot-for-whole-dataset" class="section level2" number="9.1">
<h2><span class="header-section-number">9.1</span> View Cooks distance boxplot for whole dataset</h2>
<div class="sourceCode" id="cb49"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb49-1"><a href="whole-dataset-normalisation.html#cb49-1" aria-hidden="true" tabindex="-1"></a>boxplots2 <span class="ot">&lt;-</span> </span>
<span id="cb49-2"><a href="whole-dataset-normalisation.html#cb49-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">GET_boxplot_cooks_distance</span>(<span class="at">auto_DE_output =</span> full_norm)</span></code></pre></div>
<pre><code>## $Whole_data_normalisation_output</code></pre>
<p><img src="_main_files/figure-html/unnamed-chunk-25-1.png" width="960" /></p>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="ma-plots.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="rif-analysis.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/USERNAME/REPO/edit/BRANCH/08-whole_norm.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["_main.pdf"],
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
